var newsList = {
    '2020.07': {
        name: '2020.07',
        list: [{
            TITLE: `Targeted, efficient sequence insertion and replacement in rice`,
            AUTHOR: `Jiankang Zhu Lab (Shanghai Center for Plant Stress Biology and Center for Excellence in Molecular Plant Sciences, Chinese Academy of Sciences, Shanghai, China; jkzhu@purdue.edu)`,
            JOURNAL: `Nature Biotechnology`,
            DOI: `10.1038/s41587-020-0581-5`,
            ABSTRACT: `CRISPR–Cas9 methods have been applied to generate random insertions and deletions, large deletions, targeted insertions or replacements of short sequences, and precise base changes in plants. However, versatile methods for targeted insertion or replacement of long sequences and genes, which are needed for functional genomics studies and trait improvement in crops, are few and largely depend on the use of selection markers8,9,10,11. Building on methods developed in mammalian cells12, we used chemically modified donor DNA and CRISPR–Cas9 to insert sequences of up to 2,049 base pairs (bp), including enhancers and promoters, into the rice genome at an efficiency of 25%. We also report a method for gene replacement that relies on homology-directed repair, chemically modified donor DNA and the presence of tandem repeats at target sites, achieving replacement with up to 130-bp sequences at 6.1% efficiency.`
        }, {
            TITLE: `Precise, predictable multi-nucleotide deletions in rice and wheat using APOBEC–Cas9`,
            AUTHOR: `Caixia Gao Lab (College of Advanced Agricultural Sciences, University of Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)`,
            JOURNAL: `Nature Biotechnology`,
            DOI: `10.1038/s41587-020-0566-4`,
            ABSTRACT: `Short insertions and deletions can be produced in plant genomes using CRISPR-Cas editors, but reliable production of larger deletions in specific target sites has proven difficult to achieve. We report the development of a series of APOBEC-Cas9 fusion-induced deletion systems (AFIDs) that combine Cas9 with human APOBEC3A (A3A), uracil DNA-glucosidase and apurinic or apyrimidinic site lyase. In rice and wheat, AFID-3 generated deletions from 5'-deaminated C bases to the Cas9-cleavage site. Approximately one-third of deletions produced using AFID-3 in rice and wheat protoplasts (30.2%) and regenerated plants (34.8%) were predictable. We show that eAFID-3, in which the A3A in AFID-3 is replaced with truncated APOBEC3B (A3Bctd), produced more uniform deletions from the preferred TC motif to the double-strand break. AFIDs could be applied to study regulatory regions and protein domains to improve crop plants.`
        }, {
            TITLE: `DNA capture by a CRISPR-Cas9–guided adenine base editor`,
            AUTHOR: `Jennifer A. Doudna Lab (Department of Molecular and Cell Biology, University of California, Berkeley, USA; doudna@berkeley.edu)`,
            JOURNAL: `Science`,
            DOI: `10.1126/science.abb1390`,
            ABSTRACT: `CRISPR-Cas–guided base editors convert A•T to G•C, or C•G to T•A, in cellular DNA for precision genome editing. To understand the molecular basis for DNA adenosine deamination by adenine base editors (ABEs), we determined a 3.2-angstrom resolution cryo–electron microscopy structure of ABE8e in a substrate-bound state in which the deaminase domain engages DNA exposed within the CRISPR-Cas9 R-loop complex. Kinetic and structural data suggest that ABE8e catalyzes DNA deamination up to ~1100-fold faster than earlier ABEs because of mutations that stabilize DNA substrates in a constrained, transfer RNA–like conformation. Furthermore, ABE8e’s accelerated DNA deamination suggests a previously unobserved transient DNA melting that may occur during double-stranded DNA surveillance by CRISPR-Cas9. These results explain ABE8e-mediated base-editing outcomes and inform the future design of base editors.`
        }]
    },
    '2020.06': {
        name: '2020.06',
        list: [{
            TITLE: `Quantitative regulation of Waxy expression by CRISPR/Cas9-based promoter and 5'UTR-intron editing improves grain quality in rice`,
            AUTHOR: `Yaoguang Liu Lab (College of Life Sciences, South China Agricultural University, China; ygliu@scau.edu.cn)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13427`,
            ABSTRACT: `In cereal crops, grain starches are composed of different proportions of amylose and amylopectin, which determine the cooking and eating qualities. The amylose synthesis is controlled by the Waxy (Wx) gene encoding a granule bound NDP-glucose-starch glucosyltransferase. In rice (Oryza sativa L.), the varied activities of natural Wx alleles regulate different amylose contents (AC), gel consistency (GC), and pasting viscosity of grain starches; these factors together influence the grain appearance, cooking/eating quality, and starch physical characters.`
        }, {
            TITLE: `Fine-tuning the amylose content of rice by precise base editing of the Wx gene`,
            AUTHOR: `Caixia Gao Lab (College of Advanced Agricultural Sciences, University of Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13433`,
            ABSTRACT: `The genetic diversity and phenotypic variability of crop agronomic traits is valued by breeders for their benefits in crop breeding but are limited for most target traits. Genome editing has proved to be a powerful tool for quick and efficient creation of continuous beneficial genetic variation for crop breeding. The rice Waxy (Wx) gene (LOC_Os06g04200) encodes granule-bound starch synthase I (GBSSI), which determines the amylose content (AC) of endosperm by controlling amylose synthesis. This is one of the major contributors for the eating and cooking quality (ECQ) of rice, an attribute that is receiving increased attention in society because of the improvement in people’s living standards.`
        }]
    },
    '2020.05': {
        name: '2020.05',
        list: [{
            TITLE: `Engineering herbicide resistance via prime editing in rice`,
            AUTHOR: `Magdy Mahfouz Lab (Laboratory for Genome Engineering and Synthetic Biology,  King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia; Magdy.mahfouz@kaust.edu.sa)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13399`,
            ABSTRACT: `Although CRISPR‐Cas9 has revolutionized our ability to generate site‐specific double‐strand breaks, precise editing of the genome remains challenging in most eukaryotes, including plants. In plants homology‐directed repair is inefficient, limiting our ability to make precise edits of the DNA sequence. Moreover, cytosine and adenine base editors have serious drawbacks including lower efficiency, unclean edited sequence, and the possibility of off‐target mutations at other loci. Chimeric single guide RNAs (sgRNAs) can provide editing information, in RNA form, but this modality suffers from several limitations including lower efficiency, less versatility, and the need for long homology arms.`
        }, {
            TITLE: `A DMP-triggered in vivo maternal haploid induction system in the dicotyledonous Arabidopsis`,
            AUTHOR: `Shaojiang Chen Lab (College of Agronomy and Biotechnology, China Agricultural University, Beijing, China; chen368@126.com)`,
            JOURNAL: `Nature Plants`,
            DOI: `10.1038/s41477-020-0658-7`,
            ABSTRACT: `Doubled haploid technology using inducer lines carrying mutations in ZmPLA1/MTL/NLD and ZmDMP has revolutionized traditional maize breeding. ZmPLA1/MTL/NLD is conserved in monocots and has been used to extend the system from maize to other monocots, but no functional orthologue has been identified in dicots, while ZmDMP-like genes exist in both monocots and dicots4,8,9. Here, we report that loss-of-function mutations in the Arabidopsis thaliana ZmDMP-like genes AtDMP8 and AtDMP9 induce maternal haploids, with an average haploid induction rate of 2.1 ± 1.1%. In addition, to facilitate haploid seed identification in dicots, we established an efficient FAST-Red fluorescent marker-based haploid identification system that enables the identification of haploid seeds with >90% accuracy. These results show that mutations in DMP genes also trigger haploid induction in dicots. The conserved expression patterns and amino acid sequences of ZmDMP-like genes in dicots suggest that DMP mutations could be used to develop in vivo haploid induction systems in dicots.`
        }, {
            TITLE: `Site‐directed mutagenesis in bread and durum wheat via pollination by cas9/guide RNA‐transgenic maize used as haploidy inducer`,
            AUTHOR: `Jochen Kumlehn Lab (Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Gatersleben, Germany; kumlehn@ipk-gatersleben.de)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13415`,
            ABSTRACT: `Site‐directed mutagenesis facilitates the experimental validation of gene function and can speed up plant breeding by producing new genetic variability or by reproducing previously known gene variants in other than their original genetic backgrounds. However, its application is challenging in wheat owing to high genomic redundancy and highly genotype‐dependent DNA transfer methods. In wheat, large chromosomal regions are hardly amenable to meiotic recombination, which limits the potential for trait improvements.`
        }]
    },
    '2020.04': {
        name: '2020.04',
        list: [{
            TITLE: `Development of Plant Prime-Editing Systems for Precise Genome Editing`,
            AUTHOR: `Pengcheng Wei Lab (Anhui Academy of Agricultural Science, Hefei, China; weipengcheng@gmail.com)`,
            JOURNAL: `Plant Communications`,
            DOI: `10.1016/j.xplc.2020.100043`,
            ABSTRACT: `Prime-editing systems have the capability to perform efficient and precise genome editing in human cells. In this study, we first developed a plant prime editor 2 (pPE2) system and test its activity by generating a targeted mutation on an HPT-ATG reporter in rice. Our results showed that the pPE2 system could induce programmable editing at different genome sites. In transgenic T0 plants, pPE2-generated mutants occurred with 0%–31.3% frequency, suggesting that the efficiency of pPE2 varied greatly at different genomic sites and with prime-editing guide RNAs of diverse structures. To optimize editing efficiency, guide RNAs were introduced into the pPE2 system following the PE3 and PE3b strategy in human cells. However, at the genomic sites tested in this study, pPE3 systems generated only comparable or even lower editing frequencies. Furthermore, we developed a surrogate pPE2 system by incorporating the HPT-ATG reporter to enrich the prime-edited cells. The nucleotide editing was easily detected in the resistant calli transformed with the surrogate pPE2 system, presumably due to the enhanced screening efficiency of edited cells. Taken together, our results indicate that plant prime-editing systems we developed could provide versatile and flexible editing in rice genome.`
        }, {
            TITLE: `A modular cloning toolkit for genome editing in plants`,
            AUTHOR: `Vladimir Nekrasov Lab (Plant Sciences Department, Rothamsted Research, Harpenden, UK; vladimir.nekrasov@rothamsted.ac.uk)`,
            JOURNAL: `BMC Plant Biology`,
            DOI: `10.1186/s12870-020-02388-2`,
            ABSTRACT: `Here we present an expanded cloning toolkit that contains 103 modules encoding a variety of CRISPR/Cas based nucleases and their corresponding guide RNA backbones. Among other components, the toolkit includes a number of promoters that allow expression of CRISPR/Cas nucleases (or any other coding sequences) and their guide RNAs in monocots and dicots. As part of the toolkit, we present a set of modules that enable quick and facile assembly of tRNA-sgRNA polycistronic units without a PCR step involved. We also demonstrate that our tRNA sgRNA system is functional in wheat protoplasts.`
        }]
    },
    '2020.03': {
        name: '2020.03',
        list: [{
            TITLE: `Superior field performance of waxy corn engineered using CRISPR–Cas9`,
            AUTHOR: `Huirong Gao (Research and Development, Corteva Agriscience, Johnston, USA; Huirong.gao@corteva.com)`,
            JOURNAL: `Nature BIotechnology`,
            DOI: `10.1038/s41587-020-0444-0`,
            ABSTRACT: `We created waxy corn hybrids by CRISPR–Cas9 editing of a waxy allele in 12 elite inbred maize lines, a process that was more than a year faster than conventional trait introgression using backcrossing and marker-assisted selection. Field trials at 25 locations showed that CRISPR-waxy hybrids were agronomically superior to introgressed hybrids, producing on average 5.5 bushels per acre higher yield.`
        }, {
            TITLE: `Marker-free carotenoid-enriched rice generated through targeted gene insertion using CRISPR-Cas9`,
            AUTHOR: `Pamela C. Ronald Lab (Department of Plant Pathology and the Genome Center, University of California, Davis, CA, USA; pcronald@ucdavis.edu)`,
            JOURNAL: `Nature Communications`,
            DOI: `10.1038/s41467-020-14981-y`,
            ABSTRACT: `Targeted insertion of transgenes at pre-determined plant genomic safe harbors provides a desirable alternative to insertions at random sites achieved through conventional methods. Most existing cases of targeted gene insertion in plants have either relied on the presence of a selectable marker gene in the insertion cassette or occurred at low frequency with relatively small DNA fragments (<1.8 kb). Here, we report the use of an optimized CRISPR-Cas9-based method to achieve the targeted insertion of a 5.2 kb carotenoid biosynthesis cassette at two genomic safe harbors in rice. We obtain marker-free rice plants with high carotenoid content in the seeds and no detectable penalty in morphology or yield. Whole-genome sequencing reveals the absence of off-target mutations by Cas9 in the engineered plants. These results demonstrate targeted gene insertion of marker-free DNA in rice using CRISPR-Cas9 genome editing, and offer a promising strategy for genetic improvement of rice and other crops.`
        }, {
            TITLE: `Prime genome editing in rice and wheat`,
            AUTHOR: `Caixia Gao Lab (College of Advanced Agricultural Sciences, University of Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)`,
            JOURNAL: `Nature BIotechnology`,
            DOI: `10.1038/s41587-020-0455-x`,
            ABSTRACT: `Prime editors, which are CRISPR–Cas9 nickase (H840A)– reverse transcriptase fusions programmed with prime editing guide RNAs (pegRNAs), can edit bases in mammalian cells without donor DNA or double-strand breaks. We adapted prime editors for use in plants through codon, promoter, and editing-condition optimization. The resulting suite of plant prime editors enable point mutations, insertions and deletions in rice and wheat protoplasts. Regenerated prime-edited rice plants were obtained at frequencies of up to 21.8%.`
        }, {
            TITLE: `Unconstrained genome targeting with near-PAMless engineered CRISPR-Cas9 variants`,
            AUTHOR: `Benjamin P. Kleinstiver Lab (Department of Pathology, Harvard Medical School, Boston, USA; bkleinstiver@mgh.harvard.edu)`,
            JOURNAL: `Science`,
            DOI: `10.1126/science.aba8853`,
            ABSTRACT: `Manipulation of DNA by CRISPR-Cas enzymes requires the recognition of a protospacer adjacent motif (PAM), limiting target site recognition to a subset of sequences. To remove this constraint, we engineered variants of Streptococcus pyogenes Cas9 (SpCas9) to eliminate the NGG PAM requirement. We developed a variant named SpG capable of targeting an expanded set of NGN PAMs, and further optimized this enzyme to develop a near-PAMless SpCas9 variant named SpRY (NRN>NYN PAMs). SpRY nuclease and base-editor variants can target almost all PAMs, exhibiting robust activities on a wide range of sites with NRN PAMs in human cells and lower but substantial activity on those with NYN PAMs. Using SpG and SpRY, we generated previously inaccessible disease-relevant genetic variants, supporting the utility of high-resolution targeting across genome editing applications.`
        }]
    },
    '2020.02': {
        name: '2020.02',
        list: [{
            TITLE: `Continuous evolution of SpCas9 variants compatible with non-G PAMs`,
            AUTHOR: `David R. Liu Lab (Department of Chemistry and Chemical Biology, Harvard University, Cambridge, USA; drliu@fas.harvard.edu)`,
            JOURNAL: `Nature Biotechnology`,
            DOI: `10.1038/s41587-020-0412-8`,
            ABSTRACT: `The targeting scope of Streptococcus pyogenes Cas9 (SpCas9) and its engineered variants is largely restricted to protospacer adjacent motif (PAM) sequences containing G bases. Here we report the evolution of three new SpCas9 variants that collectively recognize NRNH PAMs (where R is A or G and H is A, C or T) using phage-assisted non-continuous evolution, three new phage-assisted continuous evolution strategies for DNA binding and a secondary selection for DNA cleavage. The targeting capabilities of these evolved variants and SpCas9-NG were characterized in HEK293T cells using a library of 11,776 genomically integrated protospacer–sgRNA pairs containing all possible NNNN PAMs. The evolved variants mediated indel formation and base editing in human cells and enabled A•T-to-G•C base editing of a sickle cell anemia mutation using a previously inaccessible CACC PAM. These new evolved SpCas9 variants, together with previously reported variants, in principle enable targeting of most NR PAM sequences and substantially reduce the fraction of genomic sites that are inaccessible by Cas9-based methods.`
        }, {
            TITLE: `High-Throughput CRISPR/Cas9 Mutagenesis Streamlines Trait Gene Identification in Maize`,
            AUTHOR: `Jianbing Yan Lab (Huazhong Agricultural University, Wuhan, China; yjianbing@mail.hzau.edu.cn)`,
            JOURNAL: `Plant Cell`,
            DOI: `10.1105/tpc.19.00934`,
            ABSTRACT: `Maize is one of the most important crops in the world. However, few agronomically important maize genes have been cloned and used for trait improvement, due to the complex genome and genetic architecture. Here we integrated multiplexed CRISPR/Cas9-based high-throughput targeted mutagenesis with genetic mapping and genomic approaches to successfully knock-out 743 candidate genes corresponding to agronomic and nutritional traits. After low-cost barcode-based deep sequencing, 412 edited sequences covering 118 genes were precisely identified from individuals showing clear phenotypic changes. This mutant profile was similar to the ones identified in human cell lines and predictable. An unexpectedly frequent homology-directed repair through endogenous templates was observed and likely caused by the spatial contact between distinct chromosomes. Through several case studies on validation and interpretation of gene function, this targeted mutagenesis library suggested that the integration of forward- and reverse- genetics promises rapid validation of important agronomic genes for crops with complex genomes. Beyond specific findings, this study also guides further optimization of high-throughput CRISPR experiments in plants.`
        }]
    },
    '2020.01': {
        name: '2020.01',
        list: [{
            TITLE: `Targeted base editing in rice with CRISPR/ScCas9 system`,
            AUTHOR: `Huanbin Zhou Lab (Institute of Plant Protection, Chinese Academy of Agricultural Sciences, Beijing, China; hbzhou@ippcaas.cn)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13330`,
            ABSTRACT: `The CRISPR/Cas system has rapidly become the preferred tool for genome engineering in various organisms due to high efficiency, specificity, simplicity and versatility. Currently, CRISPR/Cas‐mediated base editing, a novel genome editing strategy that enables irreversible nucleotide changes at target loci without double‐stranded DNA cleavage or any donor template, has been widely adopted for generating gain‐of‐function germplasms in functional genomics research and crop genetic improvement.`
        }, {
            TITLE: `Targeted, random mutagenesis of plant genes with dual cytosine and adenine base editors`,
            AUTHOR: `Caixia Gao Lab (Innovation Academy for Seed Design, Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)`,
            JOURNAL: `Nature Biotechnology`,
            DOI: `10.1038/s41587-019-0393-7`,
            ABSTRACT: `Targeted saturation mutagenesis of crop genes could be applied to produce genetic variants with improved agronomic performance. However, tools for directed evolution of plant genes, such as error-prone PCR or DNA shuffling, are limited. We engineered five saturated targeted endogenous mutagenesis editors (STEMEs) that can generate de novo mutations and facilitate directed evolution of plant genes. In rice protoplasts, STEME-1 edited cytosine and adenine at the same target site with C > T efficiency up to 61.61% and simultaneous C > T and A > G efficiency up to 15.10%. STEME-NG, which incorporates the nickase Cas9-NG protospacer-adjacent motif variant, was used with 20 individual single guide RNAs in rice protoplasts to produce near-saturated mutagenesis (73.21%) for a 56-amino-acid portion of the rice acetyl-coenzyme A carboxylase (OsACC). We also applied STEME-1 and STEME-NG for directed evolution of the OsACC gene in rice and obtained herbicide resistance mutations. This set of two STEMEs will accelerate trait development and should work in any plants amenable to CRISPR-based editing.`
        }, {
            TITLE: `A Biotechnology‐Based Male Sterility System for Hybrid Seed Production in Tomato`,
            AUTHOR: `Chuanyou Li Lab (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China; cyli@genetics.ac.cn)`,
            JOURNAL: `The Plant Journal`,
            DOI: `10.1111/tpj.14678`,
            ABSTRACT: `Incorporating male sterility into hybrid seed production reduces its cost and ensures high varietal purity. Despite these advantages, male-sterile lines have not been widely used to produce tomato (Solanum lycopersicum) hybrid seeds. We describe the development of a biotechnology-based breeding platform that utilized genic male sterility to produce hybrid seeds. In this platform, we generated a novel male-sterile tomato line by clustered regularly interspaced short palindromic repeat (CRISPR)/CRISPR-associated protein 9 (Cas9)-mediated mutagenesis of a stamen-specific gene SlSTR1 and devised a transgenic maintainer by transforming male-sterile plants with a fertility-restoration gene linked to a seedling-colour gene. Offspring of crosses between a hemizygous maintainer and the homozygous male-sterile plant segregated into 50% non-transgenic male-sterile plants and 50% male-fertile maintainer plants, which could be easily distinguished by seedling colour. This system has great practical potential for hybrid seed breeding and production as it overcomes the problems intrinsic to other male sterility systems and can be easily adapted for a range of tomato cultivars and diverse vegetable crops.`
        }, {
            TITLE: `Shortened snRNA promoters for efficient CRISPR/Cas-based multiplex genome editing in monocot plants`,
            AUTHOR: `Yao-Guang Liu Lab (College of Life Sciences, South China Agricultural University, Guangzhou 510642, China; ygliu@scau.edu.cn)`,
            JOURNAL: `Science China Life Sciences`,
            DOI: `10.1007/s11427-019-1612-6`,
            ABSTRACT: `We developed a CRISPR/Cas9 multiplex genome editing system (using Agrobacterium-mediated transformation) incorporating plant snRNA promoters (OsU3, OsU6a, OsU6b, and OsU6c from rice, and AtU3b, AtU3d, AtU6-1, and AtU6-29 from Arabidopsis), which can be used for monocot and dicot plants. The rice snRNA promoters used are 380–740 bp, so the generated sgRNA expression cassettes are 510–870 bp. Using this system we assembled multiple expression cassettes by Golden Gate cloning into a vector by single Bsa I-cutting/ligation reactions, but cloning more than 8 sgRNA cassettes by this method was problematic. We reasoned that use of shorter promoters would permit linking greater numbers of sgRNA expression cassettes per round of cloning.`
        }]
    },
    '2019.12': {
        name: '2019.12',
        list: [{
            TITLE: `Using CRISPR-Cas9 to generate semi-dwarf rice lines in elite landraces`,
            AUTHOR: `Qian Qian Lab (State Key Laboratory of Rice Biology, China National Rice Research Institute, China; qianqian188@hotmail.com)`,
            JOURNAL: `Scientific Reports`,
            DOI: `10.1038/s41598-019-55757-9`,
            ABSTRACT: `Genetic erosion refers to the loss of genetic variation in a crop. In China, only a few original landraces of rice (Oryza sativa) were used in breedingand these became the primary genetic background of modern varieties. Expanding the genetic diversity among Chinese rice varieties and cultivating high-yielding and high-quality varieties with resistance to different biotic and abiotic stresses is critical. Here, we used the clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein9 (Cas9) genome editing system to edit Semi-Dwarf1 (SD1) in the elite landraces Kasalath and TeTePu (TTP), which contain many desired agronomic traits such as tolerance to low phosphorous and broad-spectrum resistance to several diseases and insects. Mutations of SD1 confer shorter plant height for better resistance to lodging. Field trials demonstrated that the yield of the new Kasalath and TTP mutant lines was better than that of the wild type under modern cultivation and that the lines maintained the same desirable agronomic characteristics as their wild-type progenitors. Our results showed that breeding using available landraces in combination with genomic data of different landraces and gene-editing techniques is an effective way to relieve genetic erosion in modern rice varieties.`
        }, {
            TITLE: `Plant gene editing through de novo induction of meristems`,
            AUTHOR: `Daniel F. Voytas Lab (Department of Genetics, Cell Biology and Development, University of Minnesota, USA; voytas@umn.edu)`,
            JOURNAL: `Nature Biotechnology`,
            DOI: `10.1038/s41587-019-0337-2`,
            ABSTRACT: `Plant gene editing is typically performed by delivering reagents such as Cas9 and single guide RNAs to explants in culture. Edited cells are then induced to differentiate into whole plants by exposure to various hormones. The creation of edited plants through tissue culture is often inefficient, time-consuming, works for only limited species and genotypes, and causes unintended changes to the genome and epigenome. Here we report two methods to generate gene-edited dicotyledonous plants through denovo meristem induction. Developmental regulators and gene-editing reagents are delivered to somatic cells of whole plants. This induces meristems that produce shoots with targeted DNA modifications, and gene edits are transmitted to the next generation. The de novo induction of gene-edited meristems side steps the need for tissue culture and promises to overcome a bottleneck in plant gene editing.`
        }, {
            TITLE: `CRISPR-TSKO: A Technique for Efficient Mutagenesis in Specific Cell Types, Tissues, or Organs in Arabidopsis`,
            AUTHOR: `Thomas B. Jacobs Lab (Department of Plant Biotechnology and Bioinformatics, Ghent University, Belgium)`,
            JOURNAL: `The Plant Cell`,
            DOI: `10.1105/tpc.19.00454`,
            ABSTRACT: `Detailed functional analyses of many fundamentally important plant genes via conventional loss-of-function approaches are impeded by the severe pleiotropic phenotypes resulting from these losses. In particular, mutations in genes that are required for basic cellular functions and/or reproduction often interfere with the generation of homozygous mutant plants, precluding further functional studies. To overcome this limitation, we devised a clustered regularly interspaced short palindromic repeats (CRISPR)-based tissue-specific knockout system, CRISPR-TSKO, enabling the generation of somatic mutations in particular plant cell types, tissues, and organs. In Arabidopsis (Arabidopsis thaliana), CRISPR-TSKO mutations in essential genes caused well-defined, localized phenotypes in the root cap, stomatal lineage, or entire lateral roots. The modular cloning system developed in this study allows for the efficient selection, identification, and functional analysis of mutant lines directly in the first transgenic generation. The efficacy of CRISPR-TSKO opens avenues for discovering and analyzing gene functions in the spatial and temporal contexts of plant life while avoiding the pleiotropic effects of system-wide losses of gene function.`
        }]
    },
    '2019.11': {
        name: '2019.11',
        list: [{
            TITLE: `Optimizing plant adenine base editor systems by modifying the transgene selection system`,
            AUTHOR: `Pengcheng Wei Lab (Anhui Academy of Agricultural Sciences, Hefei, China; Weipengcheng@gmail.com)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13304`,
            ABSTRACT: `Classical CRISPR‐Cas systems introduce a DNA double‐strand break (DSB) at target genomic loci. In plant, DSBs are typically repaired through the error‐prone nonhomologous end joining (NHEJ) pathway and result in small InDels. Recently, base editors (BEs), including cytosine BEs (CBEs) and adenine BEs (ABEs), were developed to introduce precise nucleotide substitutions by combining the CRISPR‐Cas system with engineered nucleotide deaminases. These BEs enable precise conversions between A∙T and G∙C pairs in the eukaryotic genome without introducing DSBs.`
        }, {
            TITLE: `Engineered Cas9 variant tools expand targeting scope of genome and base editing in rice`,
            AUTHOR: `Qinlong Zhu Lab (State Key Laboratory for Conservation and Utilization of Subtropical Agro‐Bioresources, Guangzhou, China; zhuql@scau.edu.cn)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13293`,
            ABSTRACT: `CRISPR/Cas9 systems have been widely used in functional genomics and crop genetic improvement, but the protospacer adjacent motif (PAM) sequence NGG of Streptococcus pyogenes Cas9 (SpCas9) limits its targeting scope. To expand the targetable genomic loci, several other Cas proteins and Cas9 variants with different PAM specificities have been developed, such as Cpf1 (Cas12a) with T‐rich PAM, and SpCas9 VQR and VRER variants with non‐canonical NGA and NGCA PAM sequence, respectively. Recently, two engineered SpCas9 variants, xCas9 3.7 and Cas9‐NG, expand the PAM recognition site to NG and show function efficiency in mammalian cells. Although a few tests of the two variants have been reported in plants, there are still not enough comparative data on their efficiencies on genome and base editing in plants. Furthermore, to reduce the possible off‐target effects, it is necessary to develop a new Cas9 variant with both enhanced specificity (eCas9) and altered PAM site such as NG. The eCas9 variant has neutralization of positive charges in the nt‐groove that can remarkably decrease off‐target. In this study, we developed a serial of variants including a new eCas9‐NG and investigated the editing activities of those variants (xCas9, Cas9‐NG and eCas9‐NG) with expended target scope in genome editing, cytosine base editors including CBE4, xCas9n‐CBE, Cas9n‐NG‐CBE and eCas9n‐NG‐CBE, adenine base editors including ABE7.10, xCas9n‐ABE, Cas9n‐NG‐ABE and eCas9n‐NG‐ABE in transgenic rice.`
        }, {
            TITLE: `CRISPR-Cas12a enables efficient biallelic gene targeting in rice`,
            AUTHOR: `Yiping Qi Lab (Department of Plant Science and Landscape Architecture, University of Maryland, College Park, MD, USA; Yiping@umd.edu)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13295`,
            ABSTRACT: `we here report a streamlined Cas12a system for efficient biallelic GT in rice within one generation. This study further sheds light on the SDSA mechanism of Cas12a‐mediated HDR suggesting precise DNA sequence changes should be designed near or downstream of the cleavage site away from the nearest homology arm that is extensively used in strand invasion during SDSA. Our study also reinforces the recent demonstration of Cas12a‐mediated GT in Arabidopsis suggesting that Cas12a is a promising tool for achieving precise gene replacement in plants.`
        }]
    },
    '2019.10': {
        name: '2019.10',
        list: [{
            TITLE: `Discriminated sgRNAs-based SurroGate System Greatly Enhances the Screening Efficiency of Plant Base-edited Cells`,
            AUTHOR: `Jinxiao Yang Lab (Beijing Academy of Agriculture & Forestry Sciences, Beijing, China; yangjinxiao@maizedna.org)`,
            JOURNAL: `Molecular Plant`,
            DOI: `10.1016/j.molp.2019.10.007`,
            ABSTRACT: `The development of CRISPR/Cas9-mediated base editing has made genomic modification more efficient. However, selecting genetically modified cells from millions of treated cells, especially plant cells, is still challenging. In this study, an efficient surrogate reporter system was established in rice to enrich base-edited cells based on a defective hygromycin resistance gene. After step-by-step optimization, the Discriminated sgRNAs-based SurroGate system (DisSUGs) was generated by artificially differentiating editing abilities using wild-type sgRNA to target the surrogate reporter gene and enhanced sgRNA to target endogenous sites. The DisSUGs enhanced the screening efficiency of base-edited cells by 3- to 5-fold for a PmCDA1-involved cytosine-to-tyrosine base editor (PCBE), and 2.5- to 6.5-fold for an adenine base editor (ABE) at endogenous targets. These targets showed editing efficiencies of <25% in the conventional system. The DisSUGs greatly enhanced the frequency of homozygous substitutions and expanded the activity window slightly for both a PCBE and an ABE. Analyses of the total number of single nucleotide variants from whole genome sequencing revealed that, compared with the non-enriched PCBE strategy, the DisSUGs did not alter the frequency of genome-wide sgRNA-independent off-target mutations, but slightly increased the frequency of target-dependent off-target mutations. Collectively, DisSUGs in this study greatly enhances the screening efficiency of plant base-edited cells and will be an useful system in the future application.`
        }, {
            TITLE: `Search-and-replace genome editing without double-strand breaks or donor DNA`,
            AUTHOR: `David R. Liu Lab (Department of Chemistry and Chemical Biology, Harvard University,Cambridge, USA; drliu@fas.harvard.edu),`,
            JOURNAL: `Nature`,
            DOI: `10.1038/s41586-019-1711-4`,
            ABSTRACT: `Most genetic variants that contribute to disease1 are challenging to correct efficiently and without excess byproducts2–5. Here we describe prime editing, a versatile and precise genome editing method that directly writes new genetic information into a specified DNA site using a catalytically impaired cas9 fused to an engineered reverse transcriptase, programmed with a prime editing guide rNA (pegrNA) that both specifies the target site and encodes the desired edit. We performed more than 175 edits in human cells including targeted insertions, deletions, and all 12 types of point mutation without requiring double-strand breaks or donor DNA templates. We applied prime editing in human cells to correct efficiently and with few byproducts the primary genetic causes of sickle cell disease (requiring a transversion in HBB) and tay-Sachs disease (requiring a deletion in HEXA), to install a protective transversion in PRNP, and to insert various tags and epitopes precisely into target loci. Four human cell lines and primary post-mitotic mouse cortical neurons support prime editing with varying efficiencies. Prime editing offers efficiency and product purity advantages over homology-directed repair, complementary strengths and weaknesses compared to base editing, and much lower off-target editing than cas9 nuclease at known cas9 off-target sites. Prime editing substantially expands the scope and capabilities of genome editing, and in principle could correct about 89% of known pathogenic human genetic variants.`
        }, {
            TITLE: `Overexpression of zmm28 increases maize grain yield in the field`,
            AUTHOR: `Jeffrey E. Habben lab (Keck Graduate Institute of Applied Life Sciences, San Francisco, CA; jeffrey.habben@corteva.com)`,
            JOURNAL: `PNAS`,
            DOI: `10.1073/pnas.1902593116`,
            ABSTRACT: `Increasing maize grain yield has been a major focus of both plant breeding and genetic engineering to meet the global demand for food, feed, and industrial uses. We report that increasing and extending expression of a maize MADS-box transcription factor gene, zmm28, under the control of a moderate-constitutive maize promoter, results in maize plants with increased plant growth, photosynthesis capacity, and nitrogen utilization. Molecular and biochemical characterization of zmm28 transgenic plants demonstrated that their enhanced agronomic traits are associated with elevated plant carbon assimilation, nitrogen utilization, and plant growth. Overall, these positive attributes are associated with a significant increase in grain yield relative to wild-type controls that is consistent across years, environments, and elite germplasm backgrounds.`
        }]
    },
    '2019.09': {
        name: '2019.09',
        list: [{
            TITLE: `Synthetic biology for evolutionary engineering: from perturbation of genotype to acquisition of desired phenotype`,
            AUTHOR: `Sang Woo Seo Lab (School of Chemical and Biological Engineering, Seoul National,University, South Korea; swseo@snu.ac.kr)`,
            JOURNAL: `Biotechnology for Biofuels`,
            DOI: `10.1186/s13068-019-1460-5`,
            ABSTRACT: `With the increased attention on bio-based industry, demands for techniques that enable fast and effective strain improvement have been dramatically increased. Evolutionary engineering, which is less dependent on biological information, has been applied to strain improvement. Currently, synthetic biology has made great innovations in evolutionary engineering, particularly in the development of synthetic tools for phenotypic perturbation. Furthermore, discovering biological parts with regulatory roles and devising novel genetic circuits have promoted high-throughput screening and selection. In this review, we frst briefy explain basics of synthetic biology tools for mutagenesis and screening of improved variants, and then describe how these strategies have been improved and applied to phenotypic engineering. Evolutionary engineering using advanced synthetic biology tools will enable further innovation in phenotypic engineering through the development of novel genetic parts and assembly into well-designed logic circuits that perform complex tasks.`
        }, {
            TITLE: `In vivo continuous evolution of metabolic pathways for chemical production`,
            AUTHOR: `Chong Zhang Lab (Department of Chemical Engineering, Tsinghua University, Beijing, China; chongzhang@tsinghua.edu.cn)`,
            JOURNAL: `Microbial Cell Factories`,
            DOI: `10.1186/s12934-019-1132-y`,
            ABSTRACT: `Microorganisms have long been used as chemical plant to convert simple substrates into complex molecules. Various metabolic pathways have been optimised over the past few decades, but the progresses were limited due to our finite knowledge on metabolism. Evolution is a knowledge-free genetic randomisation approach, employed to improve the chemical production in microbial cell factories. However, evolution of large, complex pathway was a great challenge. The invention of continuous culturing systems and in vivo genetic diversification technologies have changed the way how laboratory evolution is conducted, render optimisation of large, complex pathway possible. In vivo genetic diversification, phenotypic selection, and continuous cultivation are the key elements in in vivo continuous evolution, where any human intervention in the process is prohibited. This approach is crucial in highly efficient evolution strategy of metabolic pathway evolution.`
        }, {
            TITLE: `CHOPCHOP v3: expanding the CRISPR web toolbox beyond genome editing`,
            AUTHOR: `Eivind Valen Lab (Sars International Centre for Marine Molecular Biology, University of Bergen, Bergen, Norway; eivind.valen@gmail.com)`,
            JOURNAL: `Nucleic Acids Research`,
            DOI: `10.1093/nar/gkz365`,
            ABSTRACT: `The CRISPR–Cas system is a powerful genome editing tool that functions in a diverse array of organisms and cell types. The technology was initially developed to induce targeted mutations in DNA, but CRISPR–Cas has now been adapted to target nucleic acids for a range of purposes. CHOPCHOP is a web tool for identifying CRISPR–Cas single guide RNA (sgRNA) targets. In this major update of CHOPCHOP, we expand our toolbox beyond knockouts. We introduce functionality for targeting RNA with Cas13, which includes support for alternative transcript isoforms and RNA accessibility predictions. We incorporate new DNA targeting modes, including CRISPR activation/repression, targeted enrichment of loci for long-read sequencing, and prediction of Cas9 repair outcomes. Finally, we expand our results page visualization to reveal alternative isoforms and downstream ATG sites, which will aid users in avoiding the expression of truncated proteins. The CHOPCHOP web tool now supports over 200 genomes and we have released a command-line script for running larger jobs and handling unsupported genomes. CHOPCHOP v3 can be found at https://chopchop.cbu.uib.no`
        }, {
            TITLE: `Simplified adenine base editors improve adenine base editing efficiency in rice`,
            AUTHOR: `Jian‐Kang Zhu Lab (Shanghai Center for Plant Stress Biology, CAS Center of Excellence in Molecular Plant Sciences, Chinese Academy of Sciences, China; jkzhu@sibs.ac.cn)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13244`,
            ABSTRACT: `Adenine base editors (ABEs) have been exploited to introduce targeted adenine (A) to guanine (G) base conversions in various plant genomes, including rice, wheat and Arabidopsis. However, the ABEs reported thus far are all quite inefficient at many target sites in rice, which hampers their applications in plant genome engineering and crop breeding. Here, we show that unlike in the mammalian system, a simplified base editor ABE‐P1S (Adenine Base Editor‐Plant version 1 Simplified) containing the ecTadA*7.10‐nSpCas9 (D10A) fusion has much higher editing efficiency in rice compared to the widely used ABE‐P1 consisting of the ecTadA‐ecTadA*7.10‐nSpCas9 (D10A) fusion. We found that the protein expression level of ABE‐P1S is higher than that of ABE‐P1 in rice calli and protoplasts, which may explain the higher editing efficiency of ABE‐P1S in different rice varieties. Moreover, we demonstrate that the ecTadA*7.10‐nCas9 fusion can be used to improve the editing efficiency of other ABEs containing SaCas9 or the engineered SaKKH‐Cas9 variant. These more efficient ABEs will help advance trait improvements in rice and other crops.`
        }, {
            TITLE: `Gene targeting in Arabidopsis via an all-in-one strategy that uses a translational enhancer to aid Cas9 expression`,
            AUTHOR: `Daisuke Miki Lab (Shanghai Center for Plant Stress Biology and Center for Excellence in Molecular Plant Sciences, Chinese Academy of Sciences, China; miki@sibs.ac.cn)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13265`,
            ABSTRACT: `Gene targeting, i.e. precise sequence knock-in or replacement, is the ultimate goal of precision genome engineering. Although homologous recombination (HR)-mediated gene targeting is widely used in many organisms, including mammals, the low frequencies of HR in higher plants limit its utilization for gene targeting. We recently described a sequential transformation approach for efficient (5.3~9.1%) HR-mediated gene targeting in Arabidopsis.`
        }, {
            TITLE: `Expanding the base editing scope to GA and relaxed NG PAM sites by improved xCas9 system`,
            AUTHOR: `Jinxiao Yang Lab (Beijing Key Laboratory of Maize DNA Fingerprinting and Molecular Breeding, Beijing Academy of Agriculture & Forestry Sciences, China; yangjinxiao@maizedna.org)`,
            JOURNAL: `Plant Biotechnology Journal`,
            DOI: `10.1111/pbi.13259`,
            ABSTRACT: `Base editors have been developed to be powerful tools to generate precise point mutations. However, their applications are hindered by the strict canonical NGG PAM requirement of Streptococcus pyogenes Cas9 (SpCas9). Cas effectors recognizing different PAMs or relaxed PAMs have been employed to address this limitation. Recently, xCas9 and Cas9‐NG were both used to further broaden the editing scope to NG PAM sites. xCas9 showed broader PAM recognition including GAA and GAT than Cas9‐NG and has been employed in base editors in human cells.`
        }]
    },
    '2019.08': {
        name: '2019.08',
        list: [{
            TITLE: `CRISPR-Based Directed Evolution for Crop Improvement`,
            AUTHOR: `Magdy Mahfouz Lab (Laboratory for Genome Engineering and Synthetic Biology, King Abdullah University of Science and Technology, Thuwal, Saudi Arabia; Magdy.mahfouz@kaust.edu.sa.)`,
            JOURNAL: `Trends in Biotechnology`,
            DOI: `10.1016/j.tibtech.2019.08.001`,
            ABSTRACT: `Directed evolution involves generating diverse sequence variants of a gene of interest to produce a desirable trait under selective pressure. CRISPR-Cas9 (clustered regularly interspaced short palindromic repeats-CRISPR-associated protein 9) systems can be programmed to target any genomic locus and perform targeted directed evolution. Here, we discuss the opportunities and challenges of this emerging platform for targeted crop improvement.`
        }, {
            TITLE: `Reproducibility of CRISPR-Cas9 methods for generation of conditional mouse alleles: a multi-center evaluation`,
            AUTHOR: `Channabasavaiah B. Gurumurthy Lab (University of Nebraska Medical Center, Omaha, NE, USA; cgurumurthy@unmc.edu) `,
            JOURNAL: `Genomy Biology`,
            DOI: `10.1186/s13059-019-1776-2`,
            ABSTRACT: `We re-evaluate the two-donor method from a consortium of 20 laboratories across the world. The dataset constitutes 56 genetic loci, 17,887 zygotes, and 1718 live-born mice, of which only 15 (0.87%) mice contain cKO alleles. We subject the dataset to statistical analyses and a machine learning algorithm, which reveals that none of the factors analyzed was predictive for the success of this method. We test some of the newer methods that use one-donor DNA on 18 loci for which the two-donor approach failed to produce cKO alleles. We find that the one-donor methods are 10- to 20-fold more efficient than the two-donor approach. We propose that the two-donor method lacks efficiency because it relies on two simultaneous recombination events in cis, an outcome that is dwarfed by pervasive accompanying undesired editing events. The methods that use one-donor DNA are fairly efficient as they rely on only one recombination event, and the probability of correct insertion of the donor cassette without unanticipated mutational events is much higher. Therefore, one-donor methods offer higher efficiencies for the routine generation of cKO animal models.`
        }]
    },
    '2019.07': {
        name: '2019.07',
        list: [{
            TITLE: `Continuous evolution of base editors with expanded target compatibility and improved activity`,
            AUTHOR: `David R. Liu Lab (Howard Hughes Medical Institute, Harvard University, Cambridge, USA; drliu@fas.harvard.edu)`,
            JOURNAL: `Nature Biotechnology`,
            DOI: `10.1038/s41587-019-0193-0`,
            ABSTRACT: `Base editors use DNA-modifying enzymes targeted with a catalytically impaired CRISPR protein to precisely install point mutations. Here, we develop phage-assisted continuous evolution of base editors (BE-PACE) to improve their editing efficiency and target sequence compatibility. We used BE-PACE to evolve cytosine base editors (CBEs) that overcome target sequence context constraints of canonical CBEs. One evolved CBE, evoAPOBEC1-BE4max, is up to 26-fold more efficient at editing cytosine in the GC context, a disfavored context for wild-type APOBEC1 deaminase, while maintaining efficient editing in all other sequence contexts tested. Another evolved deaminase, evoFERNY, is 29% smaller than APOBEC1 and edits efficiently in all tested sequence contexts. We also evolved a CBE based on CDA1 deaminase with much higher editing efficiency at difficult target sites. Finally, we used data from evolved CBEs to illuminate the relationship between deaminase activity, base editing efficiency, editing window width and byproduct formation. These findings establish a system for rapid evolution of base editors and inform their use and improvement.`
        }, {
            TITLE: `Machine-learning-guided directed evolution for protein engineering`,
            AUTHOR: `Frances H. Arnold Lab (Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, USA; fha@cheme.caltech.edu)`,
            JOURNAL: `Nature Methods`,
            DOI: `10.1038/s41592-019-0496-6`,
            ABSTRACT: `Protein engineering through machine-learning-guided directed evolution enables the optimization of protein functions. Machine-learning approaches predict how sequence maps to function in a data-driven manner without requiring a detailed model of the underlying physics or biological pathways. Such methods accelerate directed evolution by learning from the properties of characterized variants and using that information to select sequences that are likely to exhibit improved properties. Here we introduce the steps required to build machine-learning sequence–function models and to use those models to guide engineering, making recommendations at each stage. This review covers basic concepts relevant to the use of machine learning for protein engineering, as well as the current literature and applications of this engineering paradigm. We illustrate the process with two case studies. Finally, we look to future opportunities for machine learning to enable the discovery of unknown protein functions and uncover the relationship between protein sequence and function.`
        }, {
            TITLE: `VEGAS as a Platform for Facile Directed Evolution in Mammalian Cells.`,
            AUTHOR: `Bryan L. Roth Lab (Department of Pharmacology, University of North Carolina, Chapel Hill, USA; bryan_roth@med.unc.edu)`,
            JOURNAL: `Cell`,
            DOI: `10.1016/j.cell.2019.05.051`,
            ABSTRACT: `Directed evolution, artificial selection toward designed objectives, is routinely used to develop new molecular tools and therapeutics. Successful directed molecular evolution campaigns repeatedly test diverse sequences with a designed selective pressure. Unicellular organisms and their viral pathogens are exceptional for this purpose and have been used for decades. However, many desirable targets of directed evolution perform poorly or unnaturally in unicellular backgrounds. Here, we present a system for facile directed evolution in mammalian cells. Using the RNA alphavirus Sindbis as a vector for heredity and diversity, we achieved 24-h selection cycles surpassing 10 −3 mutations per base. Selection is achieved through genetically actuated sequences internal to the host cell, thus the system’s name: viral evolution of genetically actuating sequences, or “VEGAS.” Using VEGAS, we evolve transcription factors, GPCRs, and allosteric nanobodies toward functional signaling endpoints each in less than 1 weeks’ time.`
        }]
    },
    '2019.06': {
        name: '2019.06',
        list: [{
            TITLE: 'Directed Evolution of a Cytochrome P450 Carbene Transferase for Selective Functionalization of Cyclic Compounds',
            AUTHOR: 'Frances H. Arnold Lab (Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, California, USA)',
            JOURNAL: 'Journal of the American Chemical Society',
            DOI: '10.1021/jacs.9b02931',
            ABSTRACT: 'Transfers of carbene moieties to heterocycles or cyclic alkenes to obtain C(sp2)‒H alkylation or cyclopropane products are valuable transformations for synthesis of pharmacophores and chemical building blocks. Through their readily tunable active site geometries, hemoprotein “carbene transferases” could provide an alternative to traditional transition metal catalysts by enabling heterocycle functionalizations with high chemo-, regio- and stereocontrol. However, carbene transferases accepting heterocyclic substrates are scarce; the few enzymes capable of heterocycle or cyclic internal alkene functionalization described to date are characterized by low turnovers or depend on artificially introduced, costly iridium–porphyrin cofactors. We addressed this challenge by evolving a cytochrome P450 for highly efficient carbene transfer to indoles, pyrroles, and cyclic alkenes. We first developed a spectrophotometric high-throughput screening assay based on 1-methylindole C3-alkylation that enabled rapid analysis of thousands of P450 variants and comprehensive directed evolution via random and targeted mutagenesis. This effort yielded a P450 variant with 11 amino acid substitutions and a large deletion of the non-catalytic P450 reductase domain, which chemoselectively C3-alkylates indoles with up to 470 turnovers per minute and 18,000 total turnovers. We subsequently used this optimized alkylation variant for parallel evolution towards more challenging heterocycle carbene functionalizations, including C2/C3 regioselective pyrrole alkylation, enantioselective indole alkylation with ethyl-2-diazopropanoate, and cyclic internal alkene cyclopropanation. The resulting set of efficient biocatalysts showcases the tunability of hemoproteins for highly selective functionalization of cyclic targets and the power of directed evolution to enhance the scope of new-to-nature enzyme catalysts. '
        }, {
            TITLE: 'Optimization of the turnover in artificial enzymes via directed evolution results in the coupling of protein dynamics to chemistry',
            AUTHOR: 'Steven D. Schwartz Lab (Department of Chemistry and Biochemistry, University of Arizona, Tucson, Arizona, United States; sschwartz@email.arizona.edu)',
            JOURNAL: 'Journal of the American Chemical Society',
            DOI: '10.1021/jacs.9b04515',
            ABSTRACT: 'The design of artiﬁcial enzymes is an emerging ﬁeld of research. Although progress has been made, the catalytic proﬁciency of many designed enzymes is low compared to natural enzymes. Nevertheless, recently Hilvert et al. created a series of ﬁve artiﬁcial retro-aldolase enzymes via directed evolution, with the ﬁnal variant exhibiting arate comparable to the natu rally occurring enzyme fructose 1,6 bisphosphate aldolase. We present a study of this system in atomistic detail, that elucidates the effects of mutational changes on the chemical step. Transition path sampling is used to create ensembles of reactive trajectories and committor analysis is used to identify the stochastic separatrix of each ensemble. The application of committor dis tribution analysis to constrained trajectories allows the identiﬁcation of changes in important protein motions coupled to reaction across the generated series of the artiﬁcial retro-aldolases. We observed two different reaction mechanisms and analyzed the role of the residues partici pating in the reaction coordinate of each enzyme. However, only in the most evolved variant we identiﬁed a fast motion that promotes catalysis, suggesting that this rate promoting vi bration was introduced during directed evolution. This study provides further evidence that protein dynamics must be taken into account in designing efﬁcient artiﬁcial enzymes.'
        }, {
            TITLE: 'Directed evolution of the metalloproteinase inhibitor TIMP-1 reveals that its N- and C-terminal domains cooperate in matrix metalloproteinase recognition',
            AUTHOR: 'Evette S. Radisky Lab (Department of Cancer Biology, Mayo Clinic, Griffin Building, Jacksonville; \'radisky.evette@mayo.edu)',
            JOURNAL: 'The Journal of Biological Chemistry',
            DOI: '10.1074/jbc.RA119.008321',
            ABSTRACT: 'Tissue inhibitors of metalloproteinases (TIMPs) are natural inhibitors of matrix metalloproteinases (MMPs), enzymes that contribute to cancer and many inflammatory and degenerative diseases. The TIMP N-terminal domain binds and inhibits an MMP catalytic domain, but the role of the TIMP C-terminal domain in MMP inhibition is poorly understood. Here, we employed yeast surface display for directed evolution of full length human TIMP-1 to develop MMP-3targeting ultra-binders. By simultaneously incorporating diversity into both domains, we identified TIMP-1 variants that were up to 10-fold improved in binding MMP-3 compared with wildtype TIMP-1, with inhibition constants (Ki) in the low picomolar range. Analysis of individual and paired mutations from the selected TIMP-1 variants revealed cooperative effects between distant residues located on the N- and C-terminal TIMP domains, positioned on opposite sides of the interaction interface with MMP-3. Crystal structures of MMP-3 complexes with TIMP-1 variants revealed conformational changes in TIMP-1 near the cooperative mutation sites. Affinity was strengthened by cinching of a reciprocal “tyrosine clasp” formed between the N terminal domain of TIMP-1 and proximal MMP-3 interface, and by changes in secondary structure within the TIMP-1 C-terminal domain that stabilize interdomain interactions and improve complementarity to MMP-3. Our protein engineering and structural studies provide critical insight into the cooperative function of TIMP domains and the significance of peripheral TIMP epitopes in MMP recognition. Our findings suggest new strategies to engineer TIMP proteins for therapeutic applications, and our directed evolution approach may also enable exploration of functional domain interactions in other protein systems.'
        }, {
            TITLE: 'Nicotine-free, Non-transgenic Tobacco (Nicotiana tabacum L.) Edited by CRISPR-Cas9',
            AUTHOR: 'Felix Stehle Lab (Department of Biochemical and Chemical Engineering, TU Dortmund University, Dortmund, Germany; felix.stehle@tu-dortmund.de)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13193',
            ABSTRACT: 'Worldwide approximately 1.1 billion people are smokers and more than 7 million people die from the negative effects of smoking every year. One of the main natural ingredients causing dependence on tobacco is nicotine. Tobacco with a lowered nicotine content could help people to overcome their nicotine addiction. Nicotine‐free (or nicotine reduced) cigarettes may contribute to reduce the number of smoker sand nicotine consumption, thus reducing the risk of death from tobacco use. Most genes involved in the nicotine biosynthesis in tobacco are known and well characterized. This opens the possibility to employ genetic engineering approaches to alter the alkaloid content of the plant, and in particular to reduce the nicotine content. Nicotine itself is composed of a pyrrolidine and a pyridine ring, which are synthesized in independent pathways (Figure 1 A). Recent approaches dealt with the silencing of upper pathway genes encoding the putrescine N‐methyltransferase (PMT) or A622, a phosphatidylinositol‐4‐phosphate (PIP)‐family member of NADPH‐reductases. The applied RNA silencing methods resulted either in the increased biosynthesis of other alkaloids like anatabine or were only successful in hairy root cultures and BY‐2 cells, but not in whole plants. The final oxidation step in the biosynthesis of nicotine, as well as anatabine and anabasine, is proposed to be catalyzed by flavoproteins of the berberine bridge enzyme‐like (BBL) family. The knockdown of the three most highly expressed BBL‐genes (BBLa–BBLc) by RNAi or the knockout with EMS‐induced mutations resulted in a reduction of the nicotine content without increasing the contentof other alkaloids. Recently, the BBL‐gene family in tobacco was expanded by the identification of BBLd.2 and BBLe, leading to six known isoforms. Thus, the simultaneous knockout of these BBL‐genes is a promising approach to generate a nicotine‐free tobacco plant.'
        }, {
            TITLE: 'Transposon-encoded CRISPR-Cas systems direct RNA-guided DNA integration',
            AUTHOR: 'Samuel H. Sternberg Lab (Department of Biochemistry and Molecular Biophysics, Columbia University, New York, USA; shsternberg@gmail.com)',
            JOURNAL: 'Nature',
            DOI: '10.1038/s41586-019-1323-z',
            ABSTRACT: 'Conventional CRISPR–Cas systems maintain genomic integrity by leveraging guide RNAs for the nuclease-dependent degradation of mobile genetic elements, including plasmids and viruses. Here we describe a remarkable inversion of this paradigm, in which bacterial Tn7-like transposons have co-opted nuclease-deficient CRISPR–Cas systems to catalyze RNA-guided integration of mobile genetic elements into the genome. Programmable transposition of Vibrio cholerae Tn6677 in E. coli requires CRISPR- and transposon-associated molecular machineries, including a novel co-complex between Cascade and the transposition protein TniQ. Donor DNA integration occurs in one of two possible orientations at a fixed distance downstream of target DNA sequences, and can accommodate variable length genetic payloads. Deep sequencing experiments reveal highly specific, genome-wide DNA integration across dozens of unique target sites. This work provides the first example of a fully programmable, RNA-guided integrase and lays the foundation for genomic manipulations that obviate the requirements for double-strand breaks and homology-directed repair.'
        }, {
            TITLE: 'RNA-guided DNA insertion with CRISPR-associated transposases',
            AUTHOR: 'Feng Zhang Lab (National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, USA; zhang@broadinstitute.org)',
            JOURNAL: 'Science',
            DOI: '10.1126/science.aax9181',
            ABSTRACT: 'CRISPR-Cas nucleases are powerful tools to manipulate nucleic acids; however, targeted insertion of DNA remains a challenge as it requires host cell repair machinery. Here we characterize a CRISPR-associated transposase (CAST) from cyanobacteria Scytonema hofmanni which consists of Tn7-like transposase subunits and the type V-K CRISPR effector (Cas12k). ShCAST catalyzes RNA-guided DNA transposition by unidirectionally inserting segments of DNA 60-66 bp downstream of the protospacer. ShCAST integrates DNA into unique sites in the E. coli genome with frequencies of up to 80% without positive selection. This work expands our understanding of the functional diversity of CRISPR-Cas systems and establishes a paradigm for precision DNA insertion.'
        }, {
            TITLE: 'In vitro and in vivo efficacy of anti-chikungunya virus monoclonal antibodies produced in wild-typeand glycoengineered N. benthamiana plants',
            AUTHOR: 'Qiang Chen Lab (The Biodesign Institute and School of Life Sciences, Arizona State University, Tempe, USA; qiang.chen.4@asu.edu)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13194',
            ABSTRACT: 'Chikungunya virus (CHIKV) is a mosquito-transmitted alphavirus, and its infection can cause long-term debilitating arthritis in humans. Currently, there are no licensed vaccines or therapeutics for human use to combat CHIKV infections. In this study, we explored the feasibility of using an anti-CHIKV monoclonal antibody (mAb) produced in wild-type (WT) and glycoengineered (ΔXFT) Nicotiana benthamiana plants in treating CHIKV infection in a mouse model. CHIKV mAb was efﬁciently expressed and assembled in plant leaves and enriched to homogeneity by a simple puriﬁcation scheme. While mAb produced in ΔXFT carried a single Nglycan species at the Fc domain, namely GnGn structures, WT produced mAb exhibited a mixture of N-glycans including the typical plant GnGnXF3 glycans, accompanied by incompletely processed and oligomannosidic structures. Both WT and ΔXFT plant-produced mAbs demonstrated potent in vitro neutralization activity against CHIKV. Notably, both mAb glycoforms showed in vivo efﬁcacy in a mouse model, with a slight increased efﬁcacy by the ΔXFT-produced mAbs. This is the ﬁrst report of the efﬁcacy of plant-produced mAbs against CHIKV, which demonstrates the ability of using plants as an effective platform for production of functionally active CHIKV mAbs and implies optimization of in vivo activity by controlling Fc glycosylation.'
        }]
    },
    '2019.05': {
        name: '2019.05',
        list: [{
            TITLE: 'CRISPR-guided DNA polymerases enable diversification of all nucleotides in a tunable window',
            AUTHOR: 'John e. Dueber Lab (Department of Bioengineering, University of California, Berkeley, USA; jdueber@berkeley.edu)',
            JOURNAL: 'Nature',
            DOI: '10.1038/s41586-018-0384-8',
            ABSTRACT: 'The capacity to diversify genetic codes advances our ability to understand and engineer biological systems. A method for continuously diversifying user-defined regions of a genome would enable forward genetic approaches in systems that are not amenable to efficient homology-directed oligonucleotide integration. It would also facilitate the rapid evolution of biotechnologically useful phenotypes through accelerated and parallelized rounds of mutagenesis and selection, as well as cell-lineage tracking through barcode mutagenesis. Here we present EvolvR, a system that can continuously diversify all nucleotides within a tunable window length at user-defined loci. This is achieved by directly generating mutations using engineered DNA polymerases targeted to loci via CRISPR-guided nickases. We identified nickase and polymerase variants that offer a range of targeted mutation rates that are up to 7,770,000-fold greater than rates seen in wild-type cells, and editing windows with lengths of up to 350 nucleotides. We used EvolvR to identify novel ribosomal mutations that confer resistance to the antibiotic spectinomycin. Our results demonstrate that CRISPR guided DNA polymerases enable multiplexed and continuous diversification of user-defined genomic loci, which will be useful for a broad range of basic and biotechnological applications.'
        }, {
            TITLE: 'Directed evolution using dcas9-targeted somatic hypermutation in mammalian cells',
            AUTHOR: 'Michael C Bassik Lab (Department of Genetics, Stanford University, Stanford, California, USA; bassik@stanford.edu)',
            JOURNAL: 'Nature Methods',
            DOI: '10.1038/nmeth.4038',
            ABSTRACT: 'engineering and study of protein function by directed evolution has been limited by the technical requirement to use global mutagenesis or introduce dnA libraries. here, we develop crisPr-X, a strategy to repurpose the somatic hypermutation machinery for protein engineering in situ. using catalytically inactive dcas9 to recruit variants of cytidine deaminase (Aid) with ms2-modifed sgrnAs, we can specifcally mutagenize endogenous targets with limited off-target damage. this generates diverse libraries of localized point mutations and can target multiple genomic locations simultaneously. We mutagenize GFP and select for spectrum-shifted variants, including eGFP. Additionally, we mutate the target of the cancer therapeutic bortezomib, PsmB5, and identify known and novel mutations that confer bortezomib resistance. Finally, using a hyperactive Aid variant, we mutagenize loci both upstream and downstream of transcriptional start sites. these experiments illustrate a powerful approach to create complex libraries of genetic variants in native context, which is broadly applicable to investigate and improve protein function.'
        }, {
            TITLE: 'Ex vivo evolution of human antibodies by CRISPR-X: from a naive B cell repertoire to affinity matured antibodies',
            AUTHOR: 'Xavier Saulquin Lab (Université de Nantes, Nantes, France; xavier.saulquin@univnantes.fr)',
            JOURNAL: 'BMC Biotechnology',
            DOI: '10.1186/s12896-019-0504-z',
            ABSTRACT: 'We use here the CRISPR-Cas 9 based CRISPR-X approach to target AID to antibody genes carried by expression vectors in HEK 293 cells. This directed mutagenesis approach, combined with a highly sensitive antigen-associated magnetic enrichment process, allowed rapid progressive evolution of a human antibody against the Human Leucocyte Antigen A*02:01 allele. Starting from a low affinity monoclonal antibody expressed on Ag-specific naïve blood circulating B cells, we obtained in approximately 6 weeks antibodies with a two log increase in affinity and which retained their specificity. Our strategy for in vitro affinity maturation of antibodies is applicable to virtually any antigen. It not only allows us to tap into the vast naive B cell repertoire but could also be useful when dealing with antigens that only elicit low affinity antibodies after immunization.'
        }]
    },
    '2019.04': {
        name: '2019.04',
        list: [{
            TITLE: 'Assessment of Cas12a-mediated gene editing efficiency in plants',
            AUTHOR: 'Diego Orzaez Lab (Universidad Politécnica de Valencia, Valencia, Spain; dorzaez@ibmcp.upv.es)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13113',
            ABSTRACT: 'The CRISPR/Cas12a editing system opens new possibilities for plant genome engineering. To obtain a comparative assessment of RNA-guided endonuclease (RGEN) types in plants, we adapted the CRISPR/Cas12a system to the GoldenBraid (GB) modular cloning platform and compared the efficiency of Acidaminococcus (As) and Lachnospiraceae (Lb) Cas12a variants with the previously described GB-assembled Streptococcus pyogens Cas9 (SpCas9) constructs in eight Nicotiana benthamiana loci using transient expression. All three nucleases showed drastic target-dependent differences in efficiency, with LbCas12 producing higher mutagenesis rates in five of the eight loci assayed, as estimated with the T7E1 endonuclease assay. Attempts to engineer crRNA direct repeat (DR) had little effect improving on-target efficiency for AsCas12a and resulted deleterious in the case of LbCas12a. To complete the assessment of Cas12a activity, we carried out genome editing experiments in three different model plants, namely N. benthamiana, Solanum lycopersicum and Arabidopsis thaliana. For the latter, we also re-sequenced Cas12a-free segregating T2 lines to assess possible off-target effects. Our results showed that the mutagenesis footprint of Cas12a is enriched in deletions of -10 to -2 nucleotides and included in some instances complex rearrangements in the surroundings of the target sites. We found no evidence of off-target mutations neither in related sequences or somewhere else in the genome. Collectively, this study shows that LbCas12a is a viable alternative to SpCas9 for plant genome engineering.'
        }, {
            TITLE: 'An adenine base editor with expanded targeting scope using SpCas9-NGv1 in rice',
            AUTHOR: 'Seiichi Toki Lab (National Agriculture and Food Research Organization, Owashi, Tsukuba, Ibaraki, Japan; stoki@affrc.go.jp)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13120',
            ABSTRACT: 'The CRISPR/Cas9 genome editing system induces mainly small insertions or deletions at the target site. In contrast, base editors are precise genome editing tools that enable base conversion at a target site without inducing a DNA double-strand break. Two types of base editors have now been developed:  cytosine base editors (CBEs)-cytidine deaminase fused with catalytically impaired Cas9-can efficiently convert a C-G base pair to a T-A pair, while adenine base editors (ABEs)-Cas9 nickase (nCas9) plus engineered Escherichia coli adenosine deaminase (TadA)-enable A-T to G-C conversions. Both CBEs and ABEs have been applied successfully in plants such as Arabidopsis, wheat, and rice.'
        }, {
            TITLE: 'An efficient DNA- and selectable-marker-free genome-editing system using zygotes in rice',
            AUTHOR: 'Takashi Okamoto Lab (Plant Breeding Innovation Lab oratory, RIKEN Cluster for Science, Technology and Innovation Hub, Yokohama, Japan; okamoto-takashi@tmu.ac.jp.)',
            JOURNAL: 'Nature Plants',
            DOI: '10.1038/s41477-019-0386-z',
            ABSTRACT: 'Technology involving the targeted mutagenesis of plants using programmable nucleases has been developing rapidly and has enormous potential in next-generation plant breeding. Notably, the clustered regularly interspaced short palindromic repeats (CRISPR)–CRISPR-associated protein-9 nuclease (Cas9) (CRISPR–Cas9) system has paved the way for the development of rapid and cost-effective procedures to create new mutant populations in plants. Although genome-edited plants from multiple species have been produced successfully using a method in which a Cas9–guide RNA (gRNA) expression cassette and selectable marker are integrated into the genomic DNA by Agrobacterium tumefaciens-mediated transformation or particle bombardment, CRISPR–Cas9 integration increases the chance of off-target modifications, and foreign DNA sequences cause legislative concerns about genetically modified organisms. Therefore, DNA-free genome editing has been developed, involving the delivery of preassembled Cas9–gRNA ribonucleoproteins (RNPs) into protoplasts derived from somatic tissues by polyethylene glycol–calcium (PEG–Ca2+)-mediated transfection in tobacco, Arabidopsis, lettuce, rice, Petunia, grapevine, apple and potato, or into embryo cells by biolistic bombardment in maize and wheat. However, the isolation and culture of protoplasts is not feasible in most plant species and the frequency of obtaining genome-edited plants through biolistic bombardment is relatively low. Here, we report a genome-editing system via direct delivery of Cas9–gRNA RNPs into plant zygotes. Cas9–gRNA RNPs were transfected into rice zygotes produced by in vitro fertilization of isolated gametes and the zygotes were cultured into mature plants in the absence of selection agents, resulting in the regeneration of rice plants with targeted mutations in around 14–64% of plants. This efficient plant-genome-editing system has enormous potential for the improvement of rice as well as other important crop species.'
        }, {
            TITLE: 'Generation of herbicide tolerance traits and a new selectable marker in wheat using base editing',
            AUTHOR: 'Caixia Gao Lab (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China. cxgao@genetics.ac.cn.)',
            JOURNAL: 'Nature Plants',
            DOI: '10.1038/s41477-019-0405-0',
            ABSTRACT: 'Developing herbicide-tolerant varieties by genome editing holds great promise for addressing the worsening weed problems in wheat cultivation1. Here, we generated transgene-free wheat germplasms harbouring herbicide tolerance mutations that confer tolerance to sulfonylurea-, imidazolinone- and aryloxyphenoxy propionate-type herbicides by base editing the acetolactate synthase (ALS) and acetyl-coenzyme A carboxylase genes. These stackable herbicide tolerance traits provide a potentially powerful tool for weed management. In addition, we found that base editing at the wheat ALS Pro-174 codon (TaALS-P174) endowed wheat with sufficient resistance to nicosulfuron herbicide in MS growth medium to allow selection. When the TaALS-P174 editor was coupled with editors for other targets of interest, co-editing occurred in the nicosulfuron-resistant plants, and selection for resistance in growth medium enriched the frequency of coupled targets by several-fold. This selectable co-editing system has the potential to greatly bolster adoption of base editing for crop improvement applications.'
        }, {
            TITLE: 'Optimizing Base Editors for Improved Efficiency and Expanded Editing Scope in Rice',
            AUTHOR: 'Jian‐Kang Zhu Lab (Shanghai Center for Plant Stress Biology and Center for Excellence in Molecular Plant Sciences, Chinese Academy of Sciences, Shanghai, China; jkzhu@purdue.edu)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13124',
            ABSTRACT: 'Base editors, presently including cytidine base editors (CBEs) and adenine base editors (ABEs), enable precise base alterations in the genome without inducing DNA double‐stranded breaks (DSBs). Base editors are valuable tools for precision plant molecular breeding since many agronomic traits are controlled by variations in one or few DNA bases. The early developed CBE and ABE systems, consisting of the rat cytidine deaminase APOBEC1 (rAPOBEC1) or activation‐induced cytidine deaminase (AID) PmCDA1, and the evolved tRNA adenine deaminase TadA, respectively, have been applied to many plant species.'
        }, {
            TITLE: 'Increasing the specificity of CRISPR systems with engineered RNA secondary structures',
            AUTHOR: 'Charles A. Gersbach Lab (Department of Surgery, Duke University Medical Center, Durham, NC, USA; gersbach@duke.edu)',
            JOURNAL: 'Nature Biotechnology',
            DOI: '10.1038/s41587-019-0095-1',
            ABSTRACT: 'CRISPR (clustered regularly interspaced short palindromic repeat) systems have been broadly adopted for basic science, biotechnology, and gene and cell therapy. In some cases, these bacterial nucleases have demonstrated off-target activity. This creates a potential hazard for therapeutic applications and could confound results in biological research. Therefore, improving the precision of these nucleases is of broad interest. Here we show that engineering a hairpin secondary structure onto the spacer region of single guide RNAs (hp-sgRNAs) can increase specificity by several orders of magnitude when combined with various CRISPR effectors. We first demonstrate that designed hp-sgRNAs can tune the activity of a transactivator based on Cas9 from Streptococcus pyogenes (SpCas9). We then show that hp-sgRNAs increase the specificity of gene editing using five different Cas9 or Cas12a variants. Our results demonstrate that RNA secondary structure is a fundamental parameter that can tune the activity of diverse CRISPR systems.'
        }, {
            TITLE: 'CRISPR/Cas9‐mediated functional recovery of the recessive rc allele to develop red rice',
            AUTHOR: 'Feng Wang Lab (Institute of Oceanography, Marine Biotechnology Center, Minjiang University, Fuzhou, China; wf@fjage.org)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13125',
            ABSTRACT: 'Red rice contains high levels of proanthocyanidins and anthocyanins, which have been recognized as health‐promoting nutrients. The red colouration of rice grains is controlled by two complementary genes, Rc and Rd. The RcRd genotype produces red pericarp in wild species Oryza rufipogon, whereas most cultivated rice varieties produce white grains resulted from a 14‐bp frame‐shift deletion in the seventh exon of the Rc gene. In the present study, we developed a CRISPR/Cas9‐mediated method to functionally restore the recessive rc allele through reverting the 14‐bp frame‐shift deletion to in‐frame mutations in which the deletions were in multiples of three bases, and successfully converted three elite white pericarp rice varieties into red ones. Rice seeds from T1 in‐frame Rc lines were measured for proanthocyanidins and anthocyanidins, and high accumulation levels of proanthocyanidins and anthocyanidins were observed in red grains from the mutants. Moreover, there was no significant difference between wild type and in‐frame Rc mutants in major agronomic traits, indicating that restoration of Rc function had no negative effect on important agronomic traits in rice. Given that most white pericarp rice varieties are resulted from the 14‐bp deletion in Rc, it is conceivable that our method could be applied to most white pericarp rice varieties and would greatly accelerate the breeding of new red rice varieties with elite agronomic traits. In addition, our study demonstrates an effective approach to restore recessive frame‐shift alleles for crop improvement.'
        }]
    },
    '2019.03': {
        name: '2019.03',
        list: [{
            TITLE: 'CRISPR/Cas9‐mediated knockout of Ms1 enables the rapid generation of male‐sterile hexaploid wheat lines for use in hybrid seed production',
            AUTHOR: 'Ryan Whitford Lab (The University of Adelaide, Waite Campus, Urrbrae, South Australia, Australia; ryan.whitford@adelaide.edu.au) ',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13106',
            ABSTRACT: 'The development and adoption of hybrid seed technology has led to dramatic increases in agricultural productivity. However, it has been a challenge to develop a commercially viable platform for the production of hybrid wheat (Triticum aestivum) seed due to wheat\'s strong inbreeding habit. Recently, a novel platform for commercial hybrid seed production was described. This hybridisation platform utilises nuclear male sterility to force outcrossing, and has been applied to maize and rice. With the recent molecular identification of the wheat male fertility gene Ms1, it is now possible to extend the use of this novel hybridisation platform to wheat. In this report, we used the CRISPR/Cas9 system to generate heritable, targeted mutations in Ms1. The introduction of biallelic frameshift mutations into Ms1 resulted in complete male sterility in wheat cultivars Fielder and Gladius, and several of the selected male‐sterile lines were potentially non‐transgenic. Our study demonstrates the utility of the CRISPR/Cas9 system for the rapid generation of male sterility in commercial wheat cultivars. This represents an important step towards capturing heterosis to improve wheat yields, through the production and use of hybrid seed on an industrial scale.',
        }, {
            TITLE: 'Precise gene replacement in rice by RNA transcript-templated homologous recombination',
            AUTHOR: 'Lanqin Xia Lab (Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, China; xialanqin@caas.cn)',
            JOURNAL: 'Nature Biotechnology',
            DOI: '10.1038/s41587-019-0065-7',
            ABSTRACT: 'One of the main obstacles to gene replacement in plants is efficient delivery of a donor repair template (DRT) into the nucleus for homology-directed DNA repair (HDR) of double-stranded DNA breaks. Production of RNA templates in vivo for transcript-templated HDR (TT-HDR) could overcome this problem, but primary transcripts are often processed and transported to the cytosol, rendering them unavailable for HDR. We show that coupling CRISPR-Cpf1 (CRISPR from Prevotella and Francisella 1) to a CRISPR RNA (crRNA) array flanked with ribozymes, along with a DRT flanked with either ribozymes or crRNA targets, produces primary transcripts that self-process to release the crRNAs and DRT inside the nucleus. We replaced the rice acetolactate synthase gene (ALS) with a mutated version using a DNA-free ribonucleoprotein complex that contains the recombinant Cpf1, crRNAs, and DRT transcripts. We also produced stable lines with two desired mutations in the ALS gene using TT-HDR.'
        }, {
            TITLE: 'Genome engineering in rice using Cas9 variants that recognize NG PAM sequences',
            AUTHOR: 'Jiankang Zhu Lab (Shanghai Center for Plant Stress Biology, CAS Center of Excellence in Molecular Plant Sciences, Chinese Academy of Sciences, Shanghai, China; jkzhu@sibs.ac.cn)',
            JOURNAL: 'Molecular Plant',
            DOI: '10.1016/j.molp.2019.03.009',
            ABSTRACT: 'CRISPR-Cas9 genome editing relies on sgRNA-target DNA base pairing and a short downstream PAM sequence to recognize target DNA. The strict PAM requirement hinders the applications of CRISPR-Cas9 system since it restricts the targetable sites in the genomes. xCas9 and SpCas9-NG are two recently engineered SpCas9 variants that can recognize more relaxed NG PAMs, which have great potential in addressing the issue of PAM constraint. Here, we use stable transgenic lines to evaluate the efficacies of xCas9 and SpCas9-NG in performing gene editing and base editing in rice. We found that xCas9 can efficiently induce mutations at target sites with NG and GAT PAM sequences in rice. However, base editors containing xCas9 failed to edit most of the tested target sites. SpCas9-NG exhibited a robust editing activity at sites with various NG PAMs without showing any preference for the third nucleotide after NG. In addition, we show that xCas9 and SpCas9-NG have higher specificity than SpCas9 at the CGG PAM site. We demonstrate that different forms of cytosine or adenine base editors containing SpCas9-NG can work efficiently in rice with broadened PAM compatibility. Thus, our work has yielded versatile genome engineering tools that will significantly expand the target scope in rice and other crops.'
        }, {
            TITLE: 'Improving plant genome editing with high-fidelity xCas9 and non-canonical PAM-targeting Cas9-NG',
            AUTHOR: 'Yiping Qi Lab (Department of Biotechnology, School of Life Sciences and Technology, Center for Informational Biology, University of Electronic Science and Technology of China, Chengdu, China; zhangyong916@uestc.edu.cn)',
            JOURNAL: 'Molecular Plant',
            DOI: '10.1016/j.molp.2019.03.011',
            ABSTRACT: 'Engineered SpCas9 variants, namely xCas9 and Cas9-NG, have shown promising potential on improving targeting specificity and broadening the targeting range. In this study, we evaluated these Cas9 variants in the model and crop plant, rice. We first tested xCas9 (xCas9-3.7, the most effective xCas9 variant in mammalian cells), for targeted mutagenesis at all 16 possible NGN PAM combinations in duplicates. xCas9 mostly exhibited equivalent editing efficiency to wild-type Cas9 (Cas9-WT) at the canonical NGG PAM sites, while possessing limited activity at non-canonical NGH (H=A, C, T) PAM sites. High editing efficiency of xCas9 at NGG PAMs was further demonstrated with C to T base editing by both rAPOBEC1 and PmCDA1 cytidine deaminases. With mismatched sgRNAs, we found xCas9 had improved targeting specificity over the Cas9-WT. We next tested two Cas9-NG variants, Cas9-NGv1 and Cas9-NG, for targeting NGN PAMs. Both Cas9-NG variants showed higher editing efficiency at many non-canonical NG PAM sites. They also enabled efficient editing, much more so than xCas9, at AT-rich PAM sites such as GAT, GAA and CAA. Cas9-NG variants, however, showed significant reduced activity at the canonical NGG PAM sites. In stable transgenic rice lines, we demonstrated Cas9-NG had much higher editing efficiency than Cas9-NGv1 and xCas9 at editing NG PAM sites. To expand base editing scope, we developed an efficient C to T base editing system by making fusion of Cas9-NG nickase (D10A version), PmCDA1 and UGI. Taken together, our work benchmarked xCas9 as a high-fidelity nuclease for targeting canonical NGG PAMs and Cas9-NG as a preferred variant for targeting relaxed PAMs for plant genome editing.'
        }, {
            TITLE: 'Cas9-NG greatly expands the targeting scope of genome-editing toolkit by recognizing NG and other atypical PAMs in rice',
            AUTHOR: 'Huanbin Zhou Lab (State Key Laboratory for Biology of Plant Diseases and Insect Pests, Institute of Plant Protection, Chinese Academy of Agricultural Sciences, Beijing, China; hbzhou@ippcaas.cn)',
            JOURNAL: 'Molecular Plant',
            DOI: '10.1016/j.molp.2019.03.010',
            ABSTRACT: 'The CRISPR technologies enabling precise genome manipulation are valuable for the gene function study and molecular crop breeding. However, the requirement of a protospacer adjacent motif (PAM), such as NGG, TTN, etc., for Cas protein recognition restricts the targetable genomic loci in applications. Very lately, Cas9-NG which recognizes a minimal NG PAM was reported to expand the targeting space in genome editing in human cells, but little is known about its applications in plants. Here, we evaluated the nuclease activity of Cas9-NG toward various NGN PAMs by targeting endogenous genes in transgenic rice, and revealed that Cas9-NG edits all NGG, NGA, NGT and NGC sites with impaired activities, and gene-edited plants were dominated by mono-allelic mutations. The Cas9-NG-engineered base editors were further developed and used to generate OsBZR1 gain-of-function plants, which other available Cas9-engineered base editors are not able to. Moreover, Cas9-NG-based transcriptional activator efficiently up-regulated the expression of endogenous target genes in rice. In addition, we discovered that Cas9-NG recognizes NAC, NTG, NTT, NCG apart from NG PAM. All these findings show that Cas9-NG greatly expands the targeting scope of genome-editing tools and imply great potentials for targeted genome editing, base editing and genome regulation in plants.'
        }, {
            TITLE: 'Chloroplast-selective gene delivery and expression in planta using chitosan-complexed single-walled carbon nanotube carriers',
            AUTHOR: 'Michael S. Strano Lab (Department of Chemical Engineering, Massachusetts Institute of Technology, Cambridge, MA, USA; strano@mit.edu)',
            JOURNAL: 'Nature nanotechnology',
            DOI: '10.1038/s41565-019-0375-4',
            ABSTRACT: 'Plant genetic engineering is an important tool used in current efforts in crop improvement, pharmaceutical product biosynthesis and sustainable agriculture. However, conventional genetic engineering techniques target the nuclear genome, prompting concerns about the proliferation of foreign genes to weedy relatives. Chloroplast transformation does not have this limitation, since the plastid genome is maternally inherited in most plants, motivating the need for organelle-specific and selective nanocarriers. Here, we rationally designed chitosan-complexed single-walled carbon nanotubes, utilizing the lipid exchange envelope penetration mechanism. The single-walled carbon nanotubes selectively deliver plasmid DNA to chloroplasts of different plant species without external biolistic or chemical aid. We demonstrate chloroplast-targeted transgene delivery and transient expression in mature Eruca sativa, Nasturtium officinale, Nicotiana tabacum and Spinacia oleracea plants and in isolated Arabidopsis thaliana mesophyll protoplasts. This nanoparticle-mediated chloroplast transgene delivery tool provides practical advantages over current delivery techniques as a potential transformation method for mature plants to benefit plant bioengineering and biological studies.'
        }, {
            TITLE: 'High aspect ratio nanomaterials enable delivery of functional genetic material without DNA integration in mature plants',
            AUTHOR: 'Markita P. Landry Lab (California Institute for Quantitative Biosciences, QB3, University of California, Berkeley, CA, USA; landry@berkeley.edu)',
            JOURNAL: 'Nature nanotechnology',
            DOI: '10.1038/s41565-019-0382-5',
            ABSTRACT: 'Genetic engineering of plants is at the core of sustainability efforts, natural product synthesis and crop engineering. The plant cell wall is a barrier that limits the ease and throughput of exogenous biomolecule delivery to plants. Current delivery methods either suffer from host-range limitations, low transformation efficiencies, tissue damage or unavoidable DNA integration into the host genome. Here, we demonstrate efficient diffusion-based biomolecule delivery into intact plants of several species with pristine and chemically functionalized high aspect ratio nanomaterials. Efficient DNA delivery and strong protein expression without transgene integration is accomplished in Nicotiana benthamiana (Nb), Eruca sativa (arugula), Triticum aestivum (wheat) and Gossypium hirsutum (cotton) leaves and arugula protoplasts. We find that nanomaterials not only facilitate biomolecule transport into plant cells but also protect polynucleotides from nuclease degradation. Our work provides a tool for species-independent and passive delivery of genetic material, without transgene integration, into plant cells for diverse biotechnology applications.'
        }, {
            TITLE: 'One-step genome editing of elite crop germplasm during haploid induction',
            AUTHOR: 'Qiudeng Que Lab (Seeds Research, Syngenta Crop Protection, Research Triangle Park, North Carolina, United States; qiudeng.que@syngenta.com)',
            JOURNAL: 'Nature Biotechnology',
            DOI: '10.1038/s41587-019-0038-x',
            ABSTRACT: 'Genome editing using CRISPR–Cas9 works efficiently in plant cells, but delivery of genome-editing machinery into the vast majority of crop varieties is not possible using established methods. We co-opted the aberrant reproductive process of haploid induction (HI) to induce edits in nascent seeds of diverse monocot and dicot species. Our method, named HI-Edit, enables direct genomic modification of commercial crop varieties. HI-Edit was tested in field and sweet corn using a native haploid-inducer line and extended to dicots using an engineered CENH3 HI system. We also recovered edited wheat embryos using Cas9 delivered by maize pollen. Our data indicate that a transient hybrid state precedes uniparental chromosome elimination in maize HI. Edited haploid plants lack both the haploid-inducer parental DNA and the editing machinery. Therefore, edited plants could be used in trait testing and directly integrated into commercial variety development.'
        }]
    },
    '2019.02': {
        name: '2019.02',
        list: [{
            'TITLE': 'CasX enzymes comprise a distinct family of RNA-guided genome editors',
            'AUTHOR': 'Jennifer A. Doudna Lab (Innovative Genomics Institute, University of California, Berkeley, CA, USA; doudna@berkeley.edu)',
            'JOURNAL': 'Nature',
            'DOI': '10.1038/s41586-019-0908-x',
            'ABSTRACT': 'The RNA-guided CRISPR-associated (Cas) proteins Cas9 and Cas12a provide adaptive immunity against invading nucleic acids, and function as powerful tools for genome editing in a wide range of organisms. Here we reveal the underlying mechanisms of a third, fundamentally distinct RNA-guided genome-editing platform named CRISPR–CasX, which uses unique structures for programmable double-stranded DNA binding and cleavage. Biochemical and in vivo data demonstrate that CasX is active for Escherichia coli and human genome modification. Eight cryo-electron microscopy structures of CasX in different states of assembly with its guide RNA and double-stranded DNA substrates reveal an extensive RNA scaffold and a domain required for DNA unwinding. These data demonstrate how CasX activity arose through convergent evolution to establish an enzyme family that is functionally separate from both Cas9 and Cas12a.',
        }, {
            'TITLE': 'Haplotype analysis of key genes governing grain yield and quality traits across 3K RG panel reveals scope for the development of tailor‐made rice with enhanced genetic gains',
            'AUTHOR': 'Arvind Kumar Lab (International Rice Research Institute, Metro Manila, Philippines; a.kumar@irri.org)',
            'JOURNAL': 'Plant Biotechnology Journal',
            'DOI': '10.1111/pbi.13087',
            'ABSTRACT': 'Though several genes governing various major traits have been reported in rice, their superior haplotype combinations for developing ideal variety remains elusive. In this study, haplotype analysis of 120 previously functionally characterized genes, influencing grain yield (87 genes) and grain quality (33 genes) revealed significant variations in the 3K rice genome (RG) panel. For selected genes, meta‐expression analysis using already available datasets along with co‐expression network provided insights at systems level. Also, we conducted candidate gene based association study for the 120 genes and identified 21 strongly associated genes governing 10‐grain yield and quality traits. We report superior haplotypes upon phenotyping the subset of 3K RG panel, SD1‐H8 with haplotype frequency (HF) of 30.13% in 3K RG panel, MOC1‐H9 (HF: 23.08%), IPA1‐H14 (HF: 6.64%), DEP3‐H2 (HF: 5.59%), DEP1‐H2 (HF: 37.53%), SP1‐H3 (HF: 5.05%), LAX1‐H5 (HF: 1.56%), LP‐H13 (3.64%), OSH1‐H4 (5.52%), PHD1‐H14 (HF: 15.21%), AGO7‐H15 (HF: 3.33%), ROC5‐H2 (31.42%), RSR1‐H8 (HF: 4.20%) and OsNAS3‐H2 (HF: 1.00%). For heading date, Ghd7‐H8 (HF: 3.08%), TOB1‐H10 (HF: 4.60%) flowered early, Ghd7‐H14 (HF: 42.60%), TRX1‐H9 (HF: 27.97%), OsVIL3‐H14 (HF: 1.72%) for medium duration flowering, while Ghd7‐H6 (HF: 1.65%), SNB‐H9 (HF: 9.35%) were late flowering. GS5‐H4 (HF: 65.84%) attributed slender, GS5‐H5 (HF: 29.00%), GW2‐H2 (HF: 4.13%) were medium slender and GS5‐H9 (HF: 2.15%) for bold grains. Furthermore, haplotype analysis explained possible genetic basis for superiority of selected mega‐varieties. Overall, this study suggests the possibility for developing next‐generation tailor‐made rice with superior haplotype combinations of target genes suiting future food and nutritional demands via haplotype‐based breeding.',
        }, {
            'TITLE': 'High efficiency Agrobacterium-mediated site-specific gene integration in maize utilizing the FLP-FRT recombination system',
            'AUTHOR': 'Ajith Anand Lab (Corteva AgriscienceTM, Agricultural Division of Dow DuPont, Johnston, USA; ajith.anand@pioneer.com)',
            'JOURNAL': 'Plant Biotechnology Journal',
            'DOI': ' 10.1111/pbi.13089 ',
            'ABSTRACT':   'An efficient Agrobacterium‐mediated site‐specific integration (SSI) technology using the FLP/FRT system in elite maize inbred lines is described. The system allows precise integration of a single copy of a donor DNA flanked by heterologous FRT sites into a predefined recombinant target line (RTL) containing the corresponding heterologous FRT sites. A promoter‐trap system consisting of a pre‐integrated promoter followed by an FRT site enables efficient selection of events. The efficiency of this system is dependent on several factors including Agrobacterium tumefaciens strain, expression of morphogenic genes Babyboom (Bbm) and Wuschel2 (Wus2) and choice of heterologous FRT pairs. Of the Agrobacterium strains tested, strain AGL1 resulted in higher transformation frequency than strain LBA4404 THY‐ (0.27% versus 0.05%; percent of infected embryos producing events). The addition of morphogenic genes increased transformation frequency (2.65% in AGL1; 0.65% in LBA4404 THY‐). Following further optimization, including the choice of FRT pairs, a method was developed that achieved 19% to 22.5% transformation frequency. Importantly, >50% of T0 transformants contain the desired full‐length site‐specific insertion. The frequencies reported here establish a new benchmark for generating targeted quality events compatible with commercial product development.',
        }, {
            'TITLE': 'A Novel Anti-HIV-1 Bispecific bNAb-Lectin Fusion Protein Engineered in a Plant-based Transient Expression System ',
            'AUTHOR': 'Nobuyuki Matoba Lab (University of Louisville School of Medicine, Louisville, USA; N.Matoba@louisville.edu)',
            'JOURNAL': 'Plant Biotechnology Journal',
            'DOI': ' 10.1111/pbi.13090 ',
            'ABSTRACT': 'The discovery of broadly neutralizing antibodies (bNAbs) has been a major step towards better prophylactic and therapeutic agents against human immunodeficiency virus type 1 (HIV‐1). However, effective therapy will likely require a combination of anti‐HIV agents to avoid viral evasion. One possible solution to this problem is the creation of bispecific molecules that can concurrently target two vulnerable sites providing synergistic inhibitory effects. Here, we describe the production in plants and anti‐HIV activity of a novel bispecific fusion protein consisting of the antigen‐binding fragment (Fab) of the CD4 binding site‐specific bNAb VRC01 and the antiviral lectin Avaren, which targets the glycan shield of HIV‐1 envelope (VRC01Fab‐Avaren). This combination was justified by a preliminary experiment demonstrating the synergistic HIV‐1 neutralization activity of VRC01 and Fc‐fused Avaren dimer (Avaren‐Fc). Using the GENEWARE® tobacco mosaic virus vector, VRC01Fab‐Avaren was expressed in Nicotiana benthamiana and purified using a three‐step chromatography procedure. Surface plasmon resonance and ELISA demonstrated that both the Avaren and VRC01Fab moieties retain their individual binding specificities. VRC01Fab‐Avaren demonstrated enhanced neutralizing activity against representative HIV‐1 strains from A, B and C clades, compared to equimolar combinations of VRC01Fab and Avaren. Notably, VRC01Fab‐Avaren showed significantly stronger neutralizing effects than the bivalent parent molecules VRC01 IgG and Avaren‐Fc, with IC50 values ranging from 48 to 310 pM. These results support the continued development of bispecific anti‐HIV proteins based on Avaren and bNAbs, to which plant‐based transient overexpression systems will provide an efficient protein engineering and production platform.',
        }, {
            'TITLE': 'Chloroplast-selective gene delivery and expression in planta using chitosan-complexed single-walled carbon nanotube carriers',
            'AUTHOR': 'Michael S. Strano Lab (Department of Chemical Engineering, Massachusetts Institute of Technology, Cambridge, MA, USA; strano@mit.edu)',
            'JOURNAL': 'Nature nanotechnology',
            'DOI': '10.1038/s41565-019-0375-4',
            'ABSTRACT': 'Plant genetic engineering is an important tool used in current efforts in crop improvement, pharmaceutical product biosynthesis and sustainable agriculture. However, conventional genetic engineering techniques target the nuclear genome, prompting concerns about the proliferation of foreign genes to weedy relatives. Chloroplast transformation does not have this limitation, since the plastid genome is maternally inherited in most plants, motivating the need for organelle-specific and selective nanocarriers. Here, we rationally designed chitosan-complexed single-walled carbon nanotubes, utilizing the lipid exchange envelope penetration mechanism. The single-walled carbon nanotubes selectively deliver plasmid DNA to chloroplasts of different plant species without external biolistic or chemical aid. We demonstrate chloroplast-targeted transgene delivery and transient expression in mature Eruca sativa, Nasturtium officinale, Nicotiana tabacum and Spinacia oleracea plants and in isolated Arabidopsis thaliana mesophyll protoplasts. This nanoparticle-mediated chloroplast transgene delivery tool provides practical advantages over current delivery techniques as a potential transformation method for mature plants to benefit plant bioengineering and biological studies.',
        }, {
            'TITLE': 'High aspect ratio nanomaterials enable delivery of functional genetic material without DNA integration in mature plants',
            'AUTHOR': 'Markita P. Landry Lab (California Institute for Quantitative Biosciences, QB3, University of California, Berkeley, CA, USA; landry@berkeley.edu)',
            'JOURNAL': 'Nature nanotechnology',
            'DOI': '10.1038/s41565-019-0382-5',
            'ABSTRACT': 'Genetic engineering of plants is at the core of sustainability efforts, natural product synthesis and crop engineering. The plant cell wall is a barrier that limits the ease and throughput of exogenous biomolecule delivery to plants. Current delivery methods either suffer from host-range limitations, low transformation efficiencies, tissue damage or unavoidable DNA integration into the host genome. Here, we demonstrate efficient diffusion-based biomolecule delivery into intact plants of several species with pristine and chemically functionalized high aspect ratio nanomaterials. Efficient DNA delivery and strong protein expression without transgene integration is accomplished in Nicotiana benthamiana (Nb), Eruca sativa (arugula), Triticum aestivum (wheat) and Gossypium hirsutum (cotton) leaves and arugula protoplasts. We find that nanomaterials not only facilitate biomolecule transport into plant cells but also protect polynucleotides from nuclease degradation. Our work provides a tool for species-independent and passive delivery of genetic material, without transgene integration, into plant cells for diverse biotechnology applications.',
        }]
    },
    '2019.01': {
        name: '2019.01',
        list: [{
            TITLE:   'Silencing an essential gene involved in infestation and digestion in grain aphid through plant‐mediated RNA interference generates aphid‐resistant wheat plants',
            AUTHOR: 'Lanqin Xia Lab ( Institute of Crop Sciences, Chinese Academy of Agricultural Sciences (CAAS), Beijing, China; xialanqin@caas.cn)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: ' 10.1111/pbi.13067 ',
            ABSTRACT: 'Aphids are major agricultural pests which cause significant yield losses of crop plants each year by inflicting damage, both through the direct effects of feeding and by vectoring debilitating plant viruses. In the absence of genetic plant resistance, insecticide treatments remain the main means for aphid control. However, excessive dependence on insecticides is undesirable because of the development of insecticide resistance, the potential negative effects on non‐target organisms and environmental pollution.'
        }, {
            TITLE: 'Creating highly efficient resistance against Wheat dwarf virus in barley by employing CRISPR/Cas9 system',
            AUTHOR: 'Zoltan Havelda Lab (National Agricultural Research and Innovation Centre, Gödöllő, Hungary; havelda.zoltan@abc.naik.hu)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13077',
            ABSTRACT: 'Wheat dwarf virus (WDV) is an economically important, phloem‐limited, insect‐transmitted virus belonging to the Geminiviridae family (Tholt et al., 2018). WDV strains infect both wheat and barley causing severe yield losses and the natural resistance resources are limited (Nygren et al., 2015). Direct utilization of the CRISPR/Cas9 system to inhibit geminivirus replication has been described in model plants (Zaidi et al., 2016). Here we show the direct anti‐viral utilization of the CRISPR/Ca9 system in an important crop plant, barley (Hordeum vulgare L. cv. Golden promise), to establish an effective WDV resistance.'
        }, {
            TITLE: 'Creating highly efficient resistance against Wheat dwarf virus in barley by employing CRISPR/Cas9 system',
            AUTHOR: 'Zoltan Havelda Lab (National Agricultural Research and Innovation Centre, Gödöllő, Hungary; havelda.zoltan@abc.naik.hu)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13077',
            ABSTRACT: 'Wheat dwarf virus (WDV) is an economically important, phloem‐limited, insect‐transmitted virus belonging to the Geminiviridae family (Tholt et al., 2018). WDV strains infect both wheat and barley causing severe yield losses and the natural resistance resources are limited (Nygren et al., 2015). Direct utilization of the CRISPR/Cas9 system to inhibit geminivirus replication has been described in model plants (Zaidi et al., 2016). Here we show the direct anti‐viral utilization of the CRISPR/Ca9 system in an important crop plant, barley (Hordeum vulgare L. cv. Golden promise), to establish an effective WDV resistance.'
        }, {
            TITLE: 'A strong early‐acting wound‐inducible promoter, RbPCD1pro, activates cryIAc expression within minutes of wounding to impart efficient protection against insects',
            AUTHOR: 'Aniruddha P. Sane Lab (Plant Gene Expression Lab, and Genomics and Molecular Biology Division, Lucknow, India; ap.sane@nbri.res.in)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13071',
            ABSTRACT: 'The expression of insecticidal proteins under constitutive promoters in transgenic plants is fraught with problems like developmental abnormalities, yield drag, expression in unwanted tissues and seasonal changes in expression. RbPCD1pro, a rapid, early‐acting wound‐inducible promoter from rose that is activated within 5 minutes of wounding, was isolated and characterized. Wounding increased transcript levels up to 150 and 500 folds within 5 and 20 minutes coupled with high translation as seen by histochemical GUS enzyme activity within 5‐20 minutes. RbPCD1pro was activated by both sucking and chewing insects and showed wound‐inducible expression in various aerial tissues of plants representing commercially important dicot and monocot families. The promoter showed no expression in any vegetative tissue except upon wounding. Functionality of RbPCD1pro was tested by its ability to drive expression of the insecticidal protein gene cryIAc in transgenic Arabidopsis and tomato. Strong wound‐inducible CryIAc expression was observed in both plants that increased 100‐350 fold (Arabidopsis) and 280‐600 fold (tomato) over the unwounded background within 5 minutes and over 1000‐1600 fold within 20 minutes. The unwounded background level was just 3‐6% of the CaMV35S promoter while wound‐induced expression was 5‐27 folds higher than the best CaMV35S line in just 5 minutes and 80 folds higher in 20 minutes. Transgenic plants showed strong resistance even to larger fourth instar larvae of H. armigera and no abnormalities in development and general plant growth. This is one of the earliest acting promoters with wide biotechnological application across monocot and dicot plants.'
        }, {
            TITLE: 'OsMS1 functions as a transcriptional activator to regulate programmed tapetum development and pollen exine formation in rice',
            AUTHOR: 'Shihua Cheng Lab (State Key Laboratory of Rice Biology, China National Rice Research Institute, Hangzhou, China; chengshihua@caas.cn)',
            JOURNAL: 'Plant Molecular Biology',
            DOI: '10.1007/s11103-018-0811-0',
            ABSTRACT: 'The tapetum, a hallmark tissue in the stamen, undergoes degradation triggered by PCD during post-meiotic anther development. This degradation process is indispensable for anther cuticle and pollen exine formation. Previous study has shown that PTC1 plays a critical role in the regulation of tapetal PCD. However, it remained unclear how this occurs. To further investigate the role of this gene in rice, we used CRISPR/Cas9 system to generate the homozygous mutant named as osms1, which showed complete male sterility with slightly yellow and small anthers, as well as invisible pollen grains. In addition, cytological observation revealed delayed tapetal PCD, defective pollen exine formation and a lack of DNA fragmentation according to a TUNEL analysis in the anthers of osms1 mutant. OsMS1, which encodes a PHD finger protein, was located in the nucleus of rice protoplasts and functioned as a transcription factor with transcriptional activation activity. Y2H and BiFC assays demonstrated that OsMS1 can interact with OsMADS15 and TDR INTERACTING PROTEIN2 (TIP2). It has been reported that TIP2 coordinated with TDR to modulate the expression of EAT1 and further regulated tapetal PCD in rice. Results of qPCR suggested that the expression of the genes associated with tapetal PCD and pollen wall biosynthesis, such as EAT1, AP37, AP25, OsC6 and OsC4, were significantly reduced in osms1 mutant. Taken together, our results demonstrate that the interaction of OsMS1 with known tapetal regulatory factors through its PHD finger regulates tapetal PCD and pollen exine formation in rice.'
        }, {
            TITLE: 'Clonal seeds from hybrid rice by simultaneous genome engineering of meiosis and fertilization genes',
            AUTHOR: 'Kejian Wang Lab (Chinese Academy of Agricultural Sciences, Hangzhou, China; wangkejian@caas.cn)',
            JOURNAL: 'Nature Biotechnology',
            DOI: '10.1038/s41587-018-0003-0',
            ABSTRACT: 'Heterosis, or hybrid vigor, is exploited by breeders to produce elite high-yielding crop lines, but beneficial phenotypes are lost in subsequent generations owing to genetic segregation. Clonal propagation through seeds would enable self-propagation of F1 hybrids. Here we report a strategy to enable clonal reproduction of F1 rice hybrids through seeds. We fixed the heterozygosity of F1 hybrid rice by multiplex CRISPR–Cas9 genome editing of the REC8, PAIR1 and OSD1meiotic genes to produce clonal diploid gametes and tetraploid seeds. Next, we demonstrated that editing the MATRILINEAL (MTL) gene (involved in fertilization) could induce formation of haploid seeds in hybrid rice. Finally, we combined fixation of heterozygosity and haploid induction by simultaneous editing of all four genes (REC8, PAIR1, OSD1and MTL) in hybrid rice and obtained plants that could propagate clonally through seeds. Application of our method may enable self-propagation of a broad range of elite F1 hybrid crops.'
        }]
    },
    '2018.12': {
        name: '2018.12',
        list: [{
            TITLE: 'Developing a highly efficient and wildly adaptive CRISPR‐SaCas9 toolset for plant genome editing',
            AUTHOR:   'Pengcheng Wei Lab (Anhui Academy of Agricultural Sciences, Hefei, China; Weipengcheng@gmail.com)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13047 ',
            ABSTRACT: 'In this study, we developed a series of mutagenesis and base editing tools using SaCas9 and its derivative for plant genome editing. These tools expand the scope of genome editing to targets with a NNNRRT PAM. It is desired to simultaneously perform different editing events, such as base editing and targeted mutation, in a single plant. Using the SaCas9 tools, this purpose would be robustly achieved by co-transforming or stacking the previously established SpCas9 systems. Taking these results together, we established a highly efficient and wildly adaptive SaCas9 toolset that may advance plant research and accelerate crop improvement.'
        }, {
            TITLE: 'xCas9 expands the scope of genome editing with reduced efficiency in rice ',
            AUTHOR: 'Kejian Wang Lab (China National Rice Research Institute, Chinese Academy of Agricultural Sciences, Hangzhou, China; wangkejian@caas.cn )',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13053 ',
            ABSTRACT: 'CRISPR‐Cas9 systems have been widely used in animals and plants for genome editing, epigenetic modification, etc. However, the accurate recognition of target sites of CRISPR/Cas9 systems depends on the single‐guide RNA (sgRNA) corresponding to each target site and protospacer adjacent motifs (PAMs) around these sites. Cas9‐sgRNA complexes will directly eject the DNA template if no PAMs are recognized before base pairing between target sites and sgRNA. '
        }, {
            TITLE: 'Single transcript unit CRISPR 2.0 systems for robust Cas9 and Cas12a mediated plant genome editing',
            AUTHOR: 'Yong Zhang Lab (University of Electronic Science and Technology of China, Chengdu, China; zhangyong916@uestc.edu.cn) ',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13068',
            ABSTRACT: 'CRISPR-Cas9 and Cas12a are two powerful genome editing systems. Expression of CRISPR in plants is typically achieved with a mixed dual promoter system, in which Cas protein is expressed by a Pol II promoter and a guide RNA is expressed by a species-specific Pol III promoter such as U6 or U3. To achieve coordinated expression and compact vector packaging, it is desirable to express both CRISPR components under a single Pol II promoter. Previously, we demonstrated a first-generation single transcript unit (STU)-Cas9 system, STU-Cas9-RZ, which is based on hammerhead ribozyme for processing single guide RNAs (sgRNAs). In this study, we developed two new STU-Cas9 systems and one STU-Cas12a system for applications in plants, collectively called the STU CRISPR 2.0 systems. We demonstrated these systems for genome editing in rice with both transient expression and stable transgenesis. The two STU-Cas9 2.0 systems process the sgRNAs with Csy4 ribonuclease and endogenous tRNA processing system, respectively. Both STU-Cas9-Csy4 and STU-Cas9-tRNA systems showed more robust genome editing efficiencies than our first-generation STU-Cas9-RZ system and the conventional mixed dual promoter system. We further applied the STU-Cas9-tRNA system to compare two C to T base editing systems based on rAPOBEC1 and PmCDA1 cytidine deaminases. The results suggest STU-based PmCDA1 base editor system is highly efficient in rice. The STU-Cas12a system, based on Cas12a’ self-processing of a CRISPR RNA (crRNA) array, was also developed and demonstrated for expression of a single crRNA and four crRNAs. Altogether, our STU CRISPR 2.0 systems further expanded the CRISPR toolbox for plant genome editing and other applications.'
        }, {
            TITLE: 'Elucidating the genetic basis of biomass accumulation and radiation use efficiency in spring wheat and its role in yield potential ',
            AUTHOR: 'Matthew P. Reynolds Lab (Global Wheat Program, International Maize and Wheat Improvement Centre (CIMMYT), Texcoco, Mexico; m.reynolds@cgiar.org)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13052 ',
            ABSTRACT: 'One of the major challenges for plant scientists is increasing wheat (Triticum aestivum) yield potential (YP). A significant bottleneck for increasing YP is achieving increased biomass through optimization of Radiation Use Efficiency (RUE) along the crop cycle. Exotic material such as landraces and synthetic wheat has been incorporated into breeding programs in an attempt to alleviate this, however their contribution to YP is still unclear. To understand the genetic basis of biomass accumulation and RUE we applied genome‐wide association study (GWAS) to a panel of 150 elite spring wheat genotypes including many landrace and synthetically derived lines. The panel was evaluated for 31traits over two years under optimal growing conditions and genotyped using the 35K Wheat Breeders array. Marker‐trait‐association identified 94 SNPs significantly associated with yield, agronomic and phenology related traits along with RUE and final biomass (BM_PM) at various growth stages that explained 7–17% of phenotypic variation. Common SNP markers were identified for grain yield, BM_PM and RUE on chromosomes 5A and 7A. Additionally, landrace and synthetic derivative lines showed higher thousand grain weight (TGW), BM_PM and RUE but lower grain number (GM2) and harvest index (HI). Our work demonstrates the use of exotic material as a valuable resource to increase YP. It also provides markers for use in marker assisted breeding to systematically increase BM_PM, RUE and TGW and avoid the TGW/GM2 and BM_PM/HI trade‐off. Thus, achieving greater genetic gains in elite germplasm while also highlighting genomic regions and candidate genes for further study.'
        }, {
            TITLE: 'Integrating a genome-wide association study with a large-scale transcriptome analysis to predict genetic regions influencing the glycaemic index and texture in rice ',
            AUTHOR: 'Nese Sreenivasulu Lab (International Rice Research Institute, Los Baños, Laguna, Philippines; n.sreenivasulu@irri.org )',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13051 ',
            ABSTRACT: 'Reliably generating rice varieties with low glycaemic index (GI) is an important nutritional intervention given the high rates of Type II diabetes incidences in Asia where rice is staple diet. We integrated a genome‐wide association study (GWAS) with a transcriptome‐wide association study (TWAS) to determine the genetic basis of the GI in rice. GWAS utilized 305 re‐sequenced diverse indica panel comprising ~2.4 million single nucleotide polymorphisms (SNPs) enriched in genic regions. A novel association signal was detected at a synonymous SNP in exon 2 of LOC_Os05g03600 for intermediate‐to‐high GI phenotypic variation. Another major hotspot region was predicted for contributing intermediate‐to‐high GI variation, involves 26 genes on chromosome 6 (GI6.1). These set of genes included GBSSI, two hydrolase genes, genes involved in signalling and chromatin modification. The TWAS and methylome sequencing data revealed cis‐acting functionally relevant genetic variants with differential methylation patterns in the hot spot GI6.1 region, narrowing the target to 13 genes. Conversely, the promoter region of GBSSI and its alternative splicing allele (G allele of Wxa) explained the intermediate‐to‐high GI variation. A SNP (C˃T) at exon‐10 was also highlighted in the preceding analyses to influence final viscosity (FV), which is independent of amylose content/GI. The low GI line with GC haplotype confirmed soft texture, while other two low GI lines with GT haplotype were characterized as hard and cohesive. The low GI lines were further confirmed through clinical in vivo studies. Gene regulatory network analysis highlighted the role of the non‐starch polysaccharide pathway in lowering GI.'
        }, {
            TITLE: 'NF-YB1-YC12-bHLH144 complex directly activates Wx to regulate grain quality in rice (Oryza sativa L.) ',
            AUTHOR: 'Jian Zhang Lab (China National Rice Research Institute, Hangzhou, China; zhangjian@caas.cn )',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13048 ',
            ABSTRACT: 'Identification of seed development regulatory genes is the key for the genetic improvement of rice grain quality. NF‐Ys are important transcription factors, but their roles in rice grain quality control and the underlying molecular mechanism remains largely unknown. Here, we report the functional characterization a rice NF‐Y heterotrimer complex NF‐YB1‐YC12‐bHLH144, which is formed by the binding of NF‐YB1 to NF‐YC12 and then bHLH144 in a sequential order. Knock‐out of each of the complex genes resulted in alteration of grain qualities in all the mutants as well as reduced grain size in crnf‐yb1 and crnf‐yc12. RNA‐seq analysis identified 1496 genes that were commonly‐regulated by NF‐YB1 and NF‐YC12, including the key granule‐bound starch synthase gene Wx. NF‐YC12 and bHLH144 maintain NF‐YB1 stability from the degradation mediated by ubiquitin/26S proteasome, while NF‐YB1 directly binds to the “G‐box” domain of Wx promoter and activates Wx transcription, hence to regulate rice grain quality. In all, we revealed a novel grain quality regulatory pathway controlled by NF‐YB1‐YC12‐bHLH144 complex, which has great potential for rice genetic improvement.'
        }]
    },
    '2018.11': {
        name: '2018.11',
        list: [{
            TITLE: 'Dominant point mutation in a tetraspanin gene associated with field-evolved resistance of cotton bollworm to transgenic Bt cotton',
            AUTHOR: 'Yidong Wu Lab (College of Plant Protection, Nanjing Agricultural University, Nanjing, Jiangsu, China; wyd@njau.edu.cn)',
            JOURNAL: 'PNAS',
            DOI: '10.1073/pnas.1812138115',
            ABSTRACT: 'Extensive planting of crops genetically engineered to produce insecticidal proteins from the bacterium Bacillus thuringiensis (Bt) has suppressed some major pests, reduced insecticide sprays, enhanced pest control by natural enemies, and increased grower profits. However, rapid evolution of resistance in pests is reducing these benefits. Better understanding of the genetic basis of resistance to Bt crops is urgently needed to monitor, delay, and counter pest resistance. We discovered that a point mutation in a previously unknown tetraspanin gene in the cotton bollworm (Helicoverpa armigera), a devastating global pest, confers dominant resistance to Cry1Ac, the sole Bt protein produced by transgenic cotton planted in China. We found the mutation using a genome-wide association study, followed by fine-scale genetic mapping and DNA sequence comparisons between resistant and susceptible strains. CRISPR/Cas9 knockout of the tetraspanin gene restored susceptibility to a resistant strain, whereas inserting the mutation conferred 125-fold resistance in a susceptible strain. DNA screening of moths captured from 23 field sites in six provinces of northern China revealed a 100-fold increase in the frequency of this mutation, from 0.001 in 2006 to 0.10 in 2016. The correspondence between the observed trajectory of the mutation and the trajectory predicted from simulation modeling shows that the dominance of the mutation accelerated adaptation. Proactive identification and tracking of the tetraspanin mutation demonstrate the potential for genomic analysis, gene editing, and molecular monitoring to improve management of resistance. '
        }, {
            TITLE: 'Characterization of transgenic rice expressing fusion protein Cry1Ab/Vip3A for insect resistance',
            AUTHOR: 'Zhicheng Shen Lab (College of Agriculture and Biotechnology, Zhejiang University, Hangzhou, China; zcshen@zju.edu.cn)',
            JOURNAL: 'Scientific Reports',
            DOI: '10.1038/s41598-018-34104-4',
            ABSTRACT: 'Management of resistance development of insect pests is of great importance for continued utilization of Bt crop. The high-dose/refuge and pyramid (gene stacking) strategy are commonly employed to delay the evolution of insect resistance. Due to the anticipated difficulty for deployment of mandatory refuge for transgenic crops in China, where the size of farmer is quite small, stacking of genes with different modes of action is a more feasible strategy. Here we report the development of transgenic rice expressing a fusion protein of Cry1Ab and Vip3A toxin. Analysis of trypsin proteolysis suggested that the fusion protein is equivalent to the combination of Cry1Ab and Vip3A protein. The transgenic plants expressing the fusion protein were found to be highly resistant to two major rice pests, Asiatic rice borer Chilo suppressalis (Lepidoptera: Crambidae) and rice leaf folder Cnaphalocrocis medinalis (Lepidoptera: Crambidae), while their agronomic performances showed no significant difference compared to the non-transgenic recipient rice. Therefore, the transgenic rice may be utilized for rice pest control in China.'
        }, {
            TITLE: 'Maize Transformation Using the Morphogenic Genes Baby Boom and Wuschel2',
            AUTHOR: 'William Gordon-Kamm Lab (Corteva Agriscience™, Agriculture Division of DowDuPont™, Johnston, USA; william.gordon-kamm@pioneer.com)',
            JOURNAL: 'Transgenic Plants',
            DOI: '10.1007/978-1-4939-8778-8_6',
            Abstract: 'Despite the fact that maize transformation has been available for over 25 years, the technology has remained too specialized, labor-intensive, and inefficient to be useful for the majority of academic labs. Compounding this problem, future demands in maize genome engineering will likely require a step change beyond what researchers view as “traditional” maize transformation methods. Recently, we published on our use of constitutively expressed morphogenic transcription factors Baby Boom (Bbm) and Wuschel2 (Wus2) to improve maize transformation, which requires CRE-mediated excision before regeneration of healthy, fertile T0 plants. Moving beyond this first-generation system, we have developed a new expression system for Bbm and Wus2, using a non-constitutive maize phospholipid transferase protein promoter (Pltppro) driving Bbm expression and a maize auxin-inducible promoter (Axig1pro) for WUS2 expression. Using this combination of expression cassettes, abundant somatic embryos rapidly form on the scutella of Agrobacterium-transformed zygotic immature embryos. These somatic embryos are uniformly transformed and can be directly germinated into plants without a callus phase. Transformed plants are sent to the greenhouse in as little as 1 month, and these T0 plants match the seed-derived phenotype for the inbred and are fertile. T1 seeds germinate normally and have a uniformly wild-type inbred phenotype. This new system represents a rapid, user-friendly transformation process that can potentially facilitate high-throughput production of transgenic T0 plants in B73, Mo17, and the recently developed Fast-Flowering Mini-Maize.'
        }, {
            TITLE: 'Segregation of genetic chimeras generated by CRISPR/Cas9 system in rice',
            AUTHOR: 'Geupil Jang Lab (School of Biological Sciences and Technology, Chonnam National University, Korea; yk3@jnu.ac.kr)',
            JOURNAL: 'Plant Biotechnology Reports',
            DOI: '10.1007/s11816-018-0508-5',
            ABSTRACT: 'CRISPR/Cas9 system based on the adaptive immune system of bacteria is an emerging powerful technique of genome editing. Previously, we showed that transgenic rice transformed with a recombinant CRISPR/Cas9-OsJAZ9 binary plasmid display genetic chimerism in T0 generation, and the genetic chimerism includes homozygous and heterogeneous mutation of the target gene, OsJAZ9. The homozygous mutation of the T0 lines was stably transferred into the next generation, and all T1 plants of the T0 homozygous mutants were homozygous mutants. However, the fate of T1 plants with heterogeneous mutation remains poorly understood. In this study, we characterized target gene mutation in the progenies of the T0 lines with heterogeneous mutation to understand fate of the T0 lines with heterogeneous mutation. We found that some T0 lines such as lines 6 and 21 produce the T1 plants carrying homozygous mutation of OsJAZ9 through genetic segregation, while other T0 lines such as lines 3, 4 and 15 did not produce homozygous mutants in T1 generation unlike lines 6 and 21. These indicated that 40% of T0 lines with heterogeneous mutation produce the plants with homozygous mutation in T1 generation, suggesting that every T0 heterogeneous line does not produce the progenies with homozygous mutation through genetic segregation. However, because the portion of T0 lines with heterogeneous mutation (61%) is much bigger than that of T0 lines with homozygous mutation (8%) in T0 pool, these results propose that T0 heterogeneous mutants are useful to obtain CRISPR/Cas9-mediated homozygous mutants.'
        }, {
            TITLE: 'CRISPR/Cas precision: do we need to worry about off-targeting in plants?',
            AUTHOR: 'Vladimir Nekrasov Lab (Plant Sciences Department, Rothamsted Research, Harpenden, UK; vladimir.nekrasov@rothamsted.ac.uk)',
            JOURNAL: 'Plant Cell Reports',
            DOI: '10.1007/s00299-018-2355-9',
            ABSTRACT: 'The CRISPR/Cas technology has recently become the tool of choice for targeted genome modification in plants and beyond. Although CRSIPR/Cas offers a rapid and facile way of introducing changes at genomic loci of interest, its application is associated with off-targeting, i.e. introduction of unintended mutations at off-target sites within the genome, which has been reported frequently in the mammalian field. Here we summarise the current knowledge on the precision of CRISPR/Cas in plant systems and provide a summary of state-of the-art strategies for avoiding off-target mutations, as well as unintended on-target changes, in plants. These include using natural (e.g. Cas12a) or engineered (e.g. SpCas9-HF) CRISPR/Cas nucleases characterised by higher precision, as compared to the commonly used wild type SpCas9. In addition, we discuss the usage of CRISPR/Cas nucleases in the form of ribonucleoproteins (RNPs) as an option for reducing off-targeting in plants. Finally, we conclude that the most important factor for reducing CRISPR/Cas off-targeting remains careful selection of target sequences, for which we provide an overview of available online software tools and experimental guidance.'
        }, {
            TITLE: 'Conferring DNA virus resistance with high specificity in plants using virus-inducible genome-editing system',
            AUTHOR: 'CaiXia Gao Lab (University of Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)',
            JOURNAL: 'Genome Biology',
            DOI: '10.1186/s13059-018-1580-4',
            ABSTRACT: 'The CRISPR/Cas9 system has recently been engineered to confer resistance to geminiviruses in plants. However, we show here that the usefulness of this antiviral strategy is undermined by off-target effects identified by deep sequencing in Arabidopsis. We construct two virus-inducible CRISPR/Cas9 vectors that efficiently inhibit beet severe curly top virus (BSCTV) accumulation in both transient assays (Nicotiana benthamiana) and transgenic lines (Arabidopsis). Deep sequencing detects no off-target effect in candidate sites of the transgenic Arabidopsis. This kind of virus-inducible genome-editing system should be widely applicable for generating virus-resistant plants without off-target costs.'
        }]
    },
    '2018.10': {
        name: '2018.10',
        list: [{
            TITLE: 'Efficient C-to-T base editing in plants using a fusion of nCas9 and human APOBEC3A',
            AUTHOR: 'CaiXia Gao Lab (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)',
            JOURNAL: 'Nature biotechnology',
            DOI: '10.1038/nbt.4261',
            ABSTRACT: 'Base editors (BEs) have been used to create C-to-T sub stitutions in various organisms. However, editing with rat APOBEC1-based BE3 is limited to a 5-nt sequence editing window and is inefficient in GC contexts. Here, we show that a base editor fusion protein composed of Cas9 nickase and human APOBEC3A (A3A-PBE) converts cytidine to thymidine efficiently in wheat, rice and potato with a 17-nucleotide editing window at all examined sites, independent of sequence context.'
        }, {
            TITLE: 'Whole genome sequencing reveals rare off‐target mutations and considerable inherent genetic or/and somaclonal variations in CRISPR/Cas9‐edited cotton plants',
            AUTHOR: 'Shuangxia Jin Lab (National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan, China; jsx@mail.hzau.edu.cn)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13020',
            ABSTRACT: 'The CRISPR/Cas9 system has been extensively applied for crop improvement. However, our understanding of Cas9 specificity is very limited in Cas9‐edited plants. To identify on‐ and off‐target mutation in an edited crop, we described whole genome sequencing (WGS) of 14 Cas9‐edited cotton plants targeted to three genes, and 3 negative (Ne) control and 3 wild‐type (WT) plants. In total, 4,188 ‐ 6,404 unique single nucleotide polymorphisms (SNPs) and 312 ‐ 745 insertions/deletions (indels) were detected in 14 Cas9‐edited plants compared with WT, negative and cotton reference genome sequences. Since the majority of these variations lack a protospacer‐adjacent motif (PAM), we demonstrated that the most variations following Cas9‐edited are due either to somaclonal variation or/and pre‐existing/natural variation from maternal plants, but not off‐target effects. Of a total of 4,413 potential off‐target sites (allowing ≤ 5 mismatches within the 20‐bp sgRNA and 3‐bp PAM sequences), the WGS data revealed that only 4 are bona fide off‐target indel mutations, validated by Sanger sequencing. Moreover, inherent genetic variation of WT can generate novel off‐target sites and destroy PAMs, which suggested great care should be taken to design sgRNA for the minimizing of off‐target effect. These findings suggested that CRISPR/Cas9 system is highly specific for cotton plants.'
        }, {
            TITLE: 'CRISPRlnc: a manually curated database of validated sgRNAs for lncRNAs  ',
            AUTHOR: 'Changning Liu Lab (Xishuangbanna Tropical Botanical Garden, Chinese Academy of Sciences, Kunming China; liuchangning@xtbg.ac.cn)',
            JOURNAL: 'Nucleic Acids Research',
            DOI: '10.1093/nar/gky904',
            ABSTRACT: 'The CRISPR/Cas9 system, as a revolutionary genome editing tool for all areas of molecular biology, provides new opportunities for research on lncRNA’s function. However, designing a CRISPR/Cas9 single guide RNA (sgRNA) for lncRNA is not easy with an unwarrantable effectiveness. Thus, it is worthy of collecting validated sgRNAs, to assist in efficiently choosing sgRNA with an expected activity. CRISPRlnc (http://www.crisprlnc.org or http://crisprlnc.xtbg.ac.cn) is a manually curated database of validated CRISPR/Cas9 sgRNAs for lncRNAs from all species. After manually reviewing more than 200 published literature, the current version of CRISPRlnc contains 305 lncRNAs and 2102 validated sgRNAs across eight species, including mammalian, insect and plant. We handled the ID, position in the genome, sequence and functional description of these lncRNAs, as well as the sequence, protoacceptor-motif (PAM), CRISPR type and validity of their paired sgRNAs. In CRISPRlnc, we provided the tools for browsing, searching and downloading data, as well as online BLAST service and genome browse server. As the first database against the validated sgRNAs of lncRNAs, CRISPRlnc will provide a new and powerful platform to promote CRISPR/Cas9 applications for future functional studies of lncRNAs.'
        }, {
            TITLE: 'Characterization of a semidominant dwarfing PROCERA allele identified in a screen for CRISPR/Cas9-induced suppressors of loss-of-function alleles',
            AUTHOR: 'Neil Olszewski Lab (Department of Plant and Microbial Biology and the Microbial and Plant Genomics Institute, University of Minnesota, St. Paul, Minnesota, Brazil; neil@umn.edu)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13027',
            ABSTRACT:   'Gibberellin (GA) is a plant hormone that regulates important growth processes including germination, flowering, and fruit development. DELLA proteins inhibit GA responses. GA signaling increases through GA triggered destruction of the DELLA protein. Several mutants with reduced stature due to mutations that stabilize the DELLA protein are known. Many of these mutations cause deletions within the DELLA domain. Here we report several new dwarfing alleles of the tomato DELLA protein, PROCERA, generated using the CRISPR/Cas9 system to create intragenic suppressors of two pro loss‐of‐function alleles. The new mutations affected the DELLA domain and one of the alleles, pro GF8, which has a 12 amino acid insertion and 2 amino acid substitution affecting the LExLE motif of DELLA domain was further characterized. This allele is semidominant. PRO/pro GF8 plants are extreme dwarfs and do not respond to exogenous GA3. However, a minor response to exogenous GA3 was found when it was applied together with the GA biosynthesis inhibitor paclobutrazol. Germination of PRO/pro GF8 seeds was delayed and pro GF8/proGF8 seeds did not germinate without scarification.'
        }, {
            TITLE: 'CRISPR-PLANT v2: An online resource for highly specific guide RNA spacers based on improved off-target analysis',
            AUTHOR: 'Yinong Yang Lab (Department of Plant Pathology and Environmental Microbiology, The Pennsylvania State University, University Park, USA; yuy3@psu.edu)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI:   '10.1111/pbi.13025',
            ABSTRACT:   'In this study, we described how current off-target prediction tools were inaccurately developed, especially in regard to the choice of aligner and their options. We showed that we achieved an improved specificity by optimizing aligners and their options to better fit short sequences like the 20 nt spacer sequences used to guide Cas9. Based on these findings, we developed a new pipeline to perform a genome-wide analysis of specific gRNAs for seven important model and crop plants. Our new strategy provided a slightly lower but still impressive number of highly specific spacer sequences that can be used to target 64.5–92.6% of the coding sequences in the seven analyzed plant genomes (Figure 1C). We assume these spacers are of highest quality compared to previously published gRNA spacer predictions because we were able to improve detection of rare but important off-target sequences. Our results are available at http://www.genome.arizona.edu/crispr2/ and can be searched by either gene locus or region of a specific species. In addition, users can clone the analysis pipeline from https://github.com/bminkenberg/CRISPR-PLANTv2 to apply it to any genome or Cas variant of their choice.'
        }, {
            TITLE: 'Minimal PAM specificity of a highly similar SpCas9 ortholog',
            AUTHOR: 'Pranam Chatterjee Lab (Massachusetts Institute of Technology, Cambridge, USA; pranam@mit.edu)',
            JOURNAL: 'Science Advances',
            DOI: '10.1126/sciadv.aau0766',
            ABSTRACT: 'RNA-guided DNA endonucleases of the CRISPR-Cas system are widely used for genome engineering and thus have numerous applications in a wide variety of fields. CRISPR endonucleases, however, require a specific protospacer adjacent motif (PAM) flanking the target site, thus constraining their targetable sequence space. In this study, we demonstrate the natural PAM plasticity of a highly similar, yet previously uncharacterized, Cas9 from Streptococcus canis (ScCas9) through rational manipulation of distinguishing motif insertions. To this end, we report affinity to minimal 5′-NNG-3′ PAM sequences and demonstrate the accurate editing capabilities of the ortholog in both bacterial and human cells. Last, we build an automated bioinformatics pipeline, the Search for PAMs by ALignment Of Targets (SPAMALOT), which further explores the microbial PAM diversity of otherwise overlooked Streptococcus Cas9 orthologs. Our results establish that ScCas9 can be used both as an alternative genome editing tool and as a functional platform to discover novel Streptococcus PAM specificities.'
        }, {
            TITLE: 'Enhanced genome editing in rice using single transcript unit CRISPR-LbCpf1 systems',
            AUTHOR: 'Pengcheng Wei Lab (Anhui Academy of Agricultural Sciences, Hefei, China; weipengcheng@gmail.com)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.13028',
            ABSTRACT: 'A variety of sequence-specific nucleases (SSNs) have been successfully used for plant genome editing. Clustered regularly interspaced short palindromic repeat (CRISPR)-CRISPR-associated (Cas) systems, which were developed from microbial adaptive immune systems, are preferred SSN tools due to their simplicity and versatility. The majority of plant genome editing studies have exploited a CRISPR-Cas9 system from Streptococcus pyogenes (SpCas9) to achieve site-specific mutagenesis and perform fragment insertion or replacement in organisms ranging from algae to higher plants. A newly identified Cas endonuclease named Cpf1 is also considered a promising genome editing tool. A Cpf1 ortholog from Lachnospiraceae bacterium ND2006 (LbCpf1) have been identified that can be programmed to induce site-specific mutagenesis in eukaryotic cells. Some features of the Cpf1 system differ from those of Cas9, which expand the application of genome editing. Cpf1 recognizes a T-rich protospacer adjacent motif (PAM) located 5’ of the target region in contrast to the 3’-G-rich PAM favored by Cas9. Cpf1 performs staggered DNA cleavage distal to the PAM site, whereas Cas9 makes blunt cuts proximal to the PAM site. Moreover, Cpf1 requires only a single CRISPR RNA (crRNA) and processes its own crRNA array, which facilitates multiplexed genome editing. Several groups, including ours, have recently reported the successful application of the CRISPR-LbCpf1 system in plant species such as rice, Arabidopsis, soybean and tobacco. However, the efficiency was variable in these systems.  '
        }]
    },
    '2018.09': {
        name: '2018.09',
        list: [{
                TITLE: 'Changes to the core and flanking sequences of G‐box elements lead to increases and decreases in gene expression in both native and synthetic soybean promoters',
                AUTHOR: 'John J. Finer Lab (Department of Horticulture and Crop Science, The Ohio State University, Wooster, USA; finer.1@osu.edu)',
                JOURNAL: 'Plant Biotechnology Journal',
                DOI: '10.1111/pbi.13010',
                ABSTRACT: 'Cis‐regulatory elements in promoters are major determinants of binding specificity of transcription factors (TFs) for transcriptional regulation. To improve our understanding of how these short DNA sequences regulate gene expression, synthetic promoters consisting of both classical (CACGTG) and variant G‐box core sequences along with different flanking sequences derived from the promoters of three different highly expressing soybean genes, were constructed and used to regulate a green fluorescent protein (gfp) gene. Use of the classical 6‐bp G‐box provided information on the base level of GFP expression while modifications to the 2‐4 flanking bases on either side of the G‐box influenced the intensity of gene expression in both transiently transformed lima bean cotyledons and stably transformed soybean hairy roots. The proximal 2‐bp sequences on either flank of the G‐box significantly affected G‐box activity, while the distal 2‐bp flanking nucleotides also influenced gene expression albeit with a decreasing effect. Manipulation of the upstream 2‐ to 4‐bp flanking sequence of a G‐box variant (GACGTG), found in the proximal region of a relatively weak soybean glycinin promoter, significantly enhanced promoter activity using both transient and stable expression assays, if the G‐box variant was first converted into a classical G‐box (CACGTG). In addition to increasing our understanding of regulatory element composition and structure, this study shows that minimal targeted changes in native promoter sequences can lead to enhanced gene expression, and suggests that genome editing of the promoter region can result in useful and predictable changes in native gene expression.'
            },
            {
                TITLE: 'Programmed Self-Elimination of the CRISPR/Cas9 Construct Greatly Accelerates the Isolation of Edited and Transgene-Free Rice Plants',
                AUTHOR: 'Yunde Zhao Lab (College of Life Science and Technology, Huazhong Agricultural University, Wuhan, China; yundezhao@ucsd.edu)',
                JOURNAL: 'Molecular Plant ',
                DOI: '10.1016/j.molp.2018.05.005',
                ABSTRACT: 'CRISPR gene-editing technology has successfully generated targeted mutations in rice and many other plant species. Assessment of heritability and phenotypic stability of CRISPR-edited plants requires the elimination of the CRISPR construct. The presence of the CRISPR construct makes it difficult to distinguish the mutations transmitted from the previous generation from newly generated mutations by the CRISPR construct at the current generation. The existence of the CRISPR construct also greatly increases the risk of off-target effects, making phenotypic stability a concern. Moreover, removal of the transgenes is likely a prerequisite for gaining regulatory approval of CRISPR-edited crops for commercial applications. Development of an efficient and easy-to-perform technology for the removal of the CRISPR construct after DNA modifications have been completed will have a major impact on using gene-editing technology for crop improvement.'
            },
            {
                TITLE: 'Manipulating plant RNA-silencing pathways to improve the gene editing efficiency of CRISPR/Cas9 systems',
                AUTHOR: 'Jiankang Zhu Lab (Shanghai Center for Plant Stress Biology, CAS Center of Excellence in Molecular Plant Sciences, Chinese Academy of Sciences, Shanghai, China; jkzhu@sibs.ac.cn)',
                JOURNAL: 'Genome Biology',
                DOI: '10.1186/s13059-018-1529-7',
                ABSTRACT: 'Here, we evaluate the expression levels of the two CRISPR/Cas9 components and the gene editing efficiency in a set of Arabidopsis mutants involved in RNA silencing. We find that mutants defective in the post-transcriptional gene-silencing pathway display significantly higher Cas9 and sgRNA transcript levels, resulting in higher mutagenesis frequencies than wild-type controls. Accordingly, silencing of AGO1 by introduction of an AGO1-RNAi cassette into the CRISPR/Cas9 vector provides an increase in gene editing efficiency. Co-expression of the viral suppressor p19 from the tomato bushy stunt virus to suppress the plant RNA-silencing pathway shows a strong correlation between the severity of the phenotypic effects caused by p19 and the gene editing efficiency of the CRISPR/Cas9 system for two different target genes, AP1 and TT4.This system has useful practical applications in facilitating the detection of CRISPR/Cas9-induced mutations in T1 plants as well as the identification of transgene-free T2 plants by simple visual observation of the symptom severity caused by p19. Our study shows that CRISPR/Cas9 gene editing efficiency can be improved by reducing RNA silencing in plants.'
            },
            {
                TITLE: 'In vivo CRISPR editing with no detectable genome-wide off-target mutations',
                AUTHOR: 'J. Keith Joung Lab (Department of Pathology, Harvard Medical School, Boston, MA, USA; jjoung@mgh.harvard.edu)',
                JOURNAL: 'Nature',
                DOI: '10.1038/s41586-018-0500-9',
                ABSTRACT: 'CRISPR–Cas genome-editing nucleases hold substantial promise for developing human therapeutic applications but identifying unwanted off-target mutations is important for clinical translation7. A well-validated method that can reliably identify off-targets in vivo has not been described to date, which means it is currently unclear whether and how frequently these mutations occur. Here we describe ‘verification of in vivo off-targets’ (VIVO), a highly sensitive strategy that can robustly identify the genome-wide off-target effects of CRISPR–Cas nucleases in vivo. We use VIVO and a guide RNA deliberately designed to be promiscuous to show that CRISPR–Cas nucleases can induce substantial off-target mutations in mouse livers in vivo. More importantly, we also use VIVO to show that appropriately designed guide RNAs can direct efficient in vivo editing in mouse livers with no detectable off-target mutations. VIVO provides a general strategy for defining and quantifying the off-target effects of gene-editing nucleases in whole organisms, thereby providing a blueprint to foster the development of therapeutic strategies that use in vivo gene editing.'
            },
            {
                TITLE: 'A single transcription factor promotes both yield and immunity in rice',
                AUTHOR: 'Xuewei Chen Lab (Sichuan Agricultural University, Chengdu, Sichuan, China; xwchen88@163.com)',
                JOURNAL: 'Science',
                DOI: '10.1126/science.aat7675',
                ABSTRACT: 'Plant immunity often penalizes growth and yield. The transcription factor Ideal Plant Architecture 1 (IPA1) reduces unproductive tillers and increases grains per panicle, which results in improved rice yield. Here we report that higher IPA1 levels enhance immunity. Mechanistically, phosphorylation of IPA1 at amino acid Ser163 within its DNA binding domain occurs in response to infection by the fungus Magnaporthe oryzae and alters the DNA binding specificity of IPA1. Phosphorylated IPA1 binds to the promoter of the pathogen defense gene WRKY45 and activates its expression, leading to enhanced disease resistance. IPA1 returns to a nonphosphorylated state within 48 hours after infection, resuming support of the growth needed for high yield. Thus, IPA1 promotes both yield and disease resistance by sustaining a balance between growth and immunity.'
            },
            {
                TITLE: 'Establishment of a modified CRISPR/Cas9 system with increased mutagenesis frequency using the translational enhancer dMac3 and multiple guide RNAs in potato',
                AUTHOR: 'Hiroaki Shimada La (Department of Biological Science and Technology, Tokyo University of Science, Katsushika, Tokyo, Japan; shimadah@rs.noda.tus.ac.jp)',
                JOURNAL: 'Science Report',
                DOI: '10.1038/s41598-018-32049-2',
                ABSTRACT: 'CRISPR/Cas9 is a programmable nuclease composed of the Cas9 protein and a guide RNA (gRNA) molecule. To create a mutant potato, a powerful genome-editing system was required because potato has a tetraploid genome. The translational enhancer dMac3, consisting of a portion of the OsMac3 mRNA 5′-untranslated region, greatly enhanced the production of the protein encoded in the downstream ORF. To enrich the amount of Cas9, we applied the dMac3 translational enhancer to the Cas9 expression system with multiple gRNA genes. CRISPR/Cas9 systems targeting the potato granule-bound starch synthase I (GBSSI) gene examined the frequency of mutant alleles in transgenic potato plants. The efficiency of the targeted mutagenesis strongly increased when the dMac3-installed Cas9 was used. In this case, the ratio of transformants containing four mutant alleles reached approximately 25% when estimated by CAPS analysis. The mutants that exhibited targeted mutagenesis in the GBSSI gene showed characteristics of low amylose starch in their tubers. This result suggests that our system may facilitate genome-editing events in polyploid plants.'
            }
        ]
    },
    '2018.08': {
        name: '2018.08',
        list: [{
                TITLE: 'Genome editing of upstream open reading frames enables translational control in plants',
                AUTHOR: 'CaiXia Gao Lab (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)',
                JOURNAL: 'Nature Biotechnology',
                DOI: '10.1038/nbt.4202',
                ABSTRACT: 'Translational regulation by upstream open reading frames (uORFs) is becoming established as a general mechanism for controlling the amount of protein that is synthesized from downstream primary ORFs (pORFs). We found that genome editing of endogenous uORFs in plants enabled the modulation of translation of mRNAs from four pORFs that are involved in either development or antioxidant biosynthesis. A single-guide RNA that targeted the region harboring a uORF initiation codon can produce multiple mutations. Following uORF editing, we observed varying amounts of mRNA translation in four pORFs. Notably, editing the uORF of LsGGP2, which encodes a key enzyme in vitamin C biosynthesis in lettuce, not only increased oxidation stress tolerance, but also increased ascorbate content by ∼150%. These data indicate that editing plant uORFs provides a generalizable, efficient method for manipulating translation of mRNA that could be applied to dissect biological mechanisms and improve crops.'
            },
            {
                TITLE: 'Natural Variation in OsLG3 Increases Drought Tolerance in Rice by Inducing ROS Scavenging',
                AUTHOR: 'Zichao Li Lab (China Agricultural University, Beijing, China; lizichao@cau.edu.cn)',
                JOURNAL: 'Plant Physiology',
                DOI: '10.1104/pp.17.01492',
                ABSTRACT: 'Improving the performance of rice (Oryza sativa L.) under drought stress has the potential to significantly affect rice productivity. Here, we report that the ERF family transcription factor OsLG3 positively regulates drought tolerance in rice. In our previous work, we found that OsLG3 has a positive effect on rice grain length without affecting grain quality. In this study, we found that OsLG3 was more strongly expressed in upland rice than in lowland rice under drought-stress conditions. By performing candidate gene association analysis, we found that natural variation in the promoter of OsLG3 is associated with tolerance to osmotic stress in germinating rice seeds. Overexpression of OsLG3 significantly improved the tolerance of rice plants to simulated drought, whereas suppression of OsLG3 resulted in greater susceptibility. Phylogenetic analysis indicated that the tolerant allele of OsLG3 may improve drought tolerance in cultivated japonica rice. Introgression lines and complementation transgenic lines containing the elite allele of OsLG3IRAT109 showed increased drought tolerance, demonstrating that natural variation in OsLG3 contributes to drought tolerance in rice. Further investigation suggested that OsLG3 plays a positive role in drought-stress tolerance in rice by inducing reactive oxygen species scavenging. Collectively, our findings reveal that natural variation in OsLG3 contributes to rice drought tolerance and that the elite allele of OsLG3 is a promising genetic resource for the development of drought-tolerant rice varieties.'
            },
            {
                TITLE: 'Endosperm sugar accumulation caused by mutation of PHS8/ISA1 leads to pre‐harvest sprouting in rice',
                AUTHOR: 'Chengcai Chu Lab (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China; ccchu@genetics.ac.cn)',
                JOURNAL: 'The Plant Journal',
                DOI: '10.1111/tpj.13970',
                ABSTRCT: 'Pre‐harvest sprouting (PHS) is an unfavorable trait in cereal crops that could seriously decrease grain yield and quality. Although some PHS‐associated quantitative trait loci or genes in cereals have been reported, the molecular mechanism underlying PHS remains largely elusive. Here, we characterized a rice mutant, phs8, which exhibits PHS phenotype accompanied by sugary endosperm. Map‐based cloning revealed that PHS8 encodes a starch debranching enzyme named isoamylase1. Mutation in PHS8 resulted in the phytoglycogen breakdown and sugar accumulation in the endosperm. Intriguingly, with increase of sugar contents, decreased expression of OsABI3 and OsABI5 as well as reduced sensitivity to abscisic acid (ABA) were found in the phs8 mutant. Using rice suspension cell system, we confirmed that exogenous sugar is sufficient to suppress the expression of both OsABI3 and OsABI5. Furthermore, overexpression of OsABI3 or OsABI5 could partially rescue the PHS phenotype of phs8. Therefore, our study presents important evidence supporting that endosperm sugar not only acts as an essential energy source for seed germination but also determines seed dormancy and germination by affecting ABA signaling.'
            },
            {
                TITLE: 'Osa‐miR164a targets OsNAC60 and negatively regulates rice immunity against the blast fungus Magnaporthe oryzae',
                AUTHOR: 'Hongwei Zhao Lab (The Key Laboratory of Integrated Management of Crop Diseases and Pests, Ministry of Education, Nanjing, China; hzhao@njau.edu.cn)',
                JOURNAL: 'The Plant Journal',
                DOI: '10.1111/tpj.13972',
                ABSTRACT: 'Exploring the regulatory mechanism played by endogenous rice miRNAs in defense responses against the blast disease is of great significance in both resistant variety breeding and disease control management. We identified rice defense‐related miRNAs by comparing rice miRNA expression patterns before and after Magnaporthe oryzae strain Guy11 infection. We discovered that osa‐miR164a expression reduced upon Guy11 infection at both early and late stages, which was perfectly associated with the induced expression of its target gene, OsNAC60. OsNAC60 encodes a transcription factor, over‐expression of which enhanced defense responses, such as increased programmed cell death, greater ion leakage, more reactive oxygen species accumulation and callose deposition, and upregulation of defense‐related genes. By using transgenic rice over‐expressing osa‐miR164a, and a transposon insertion mutant of OsNAC60, we showed that when the miR164a/OsNAC60 regulatory module was dysfunctional, rice developed significant susceptibility to Guy11 infection. The co‐expression of OsNAC60 and osa‐miR164a abolished the OsNAC60 activity, but not its synonymous mutant. We further validated that this regulatory module is conserved in plant resistance to multiple plant diseases, such as the rice sheath blight, tomato late blight, and soybean root and stem rot diseases. Our results demonstrate that the miR164a/OsNAC60 regulatory module manipulates rice defense responses to M. oryzae infection. This discovery is of great potential for resistant variety breeding and disease control to a broad spectrum of pathogens in the future.'
            },
            {
                TITLE: 'The chimeric mitochondrial gene orf182 causes non‐pollen‐type abortion in Dongxiang cytoplasmic male‐sterile rice',
                AUTHOR: 'Yaohui Cai Lab (Jiangxi Academy of Agricultural Sciences, National Engineering Laboratory for Rice, Nanchang, China; caiyaohui@126.com)',
                JOURNAL: 'The Plant Journal',
                DOI: '10.1111/tpj.13982',
                ABSTRACT: 'D1‐cytoplasmic male sterility (CMS) rice is a sporophytic cytoplasmic male‐sterile rice developed from Dongxiang wild rice that exhibits a no‐pollen‐grain phenotype. A mitochondrial chimeric gene (orf182) was detected by mitochondrial genome sequencing and a comparative analysis. Orf182 is composed of three recombinant fragments, the largest of which is homologous to Sorghum bicolor mitochondrial sequences. In addition, orf182 was found only in wild rice species collected from China. Northern blot analysis showed that orf182 transcripts were affected by Rf genes in the isocytoplasmic restorer line DR7. Western blot analysis showed that the ORF182 product was localized in the mitochondria of the CMS line. An expression cassette containing orf182 fused to a mitochondrial transit peptide induced the maintainer line of male sterility, which lacked pollen grains in the anthers. Furthermore, the in vivo expression of orf182 also inhibited the growth of Escherichia coli, with lower respiration rate, excess accumulation of reactive oxygen species and decreased ATP levels. We conclude that the mitochondrial chimeric gene orf182 possesses a unique structure and origin differing from other identified mitochondrial CMS genes, and this gene is connected to non‐pollen type of sporophytic male sterility in D1‐CMS rice.'
            },
            {
                TITLE: 'DELAYED HEADING DATE1 interacts with OsHAP5C/D, delays flowering time and enhances yield in rice',
                AUTHOR: 'Jianmin Wang Lab (Chinese Academy of Agricultural Sciences, Beijing, China; wanjianmin@caas.cn)',
                JOURNAL: 'Plant Biotechnology Journal',
                DOI: '10.1111/pbi.12996',
                ABSTRACT: 'Heading date is an important agronomic trait affecting crop yield. The GRAS protein family is a plant‐specific super family extensively involved in plant growth and signal transduction. However, GRAS proteins are rarely reported have a role in regulating rice heading date. Here, we report a GRAS protein DHD1 (Delayed Heading Date1) delays heading and enhances yield in rice. Biochemical assays showed DHD1 physically interacts with OsHAP5C/D both in vitro and in vivo. DHD1 and OsHAP5C/D located in the nucleus and showed that rhythmic expression. Both DHD1 and OsHAP5C/D affect heading date by regulating expression of Ehd1. We propose that DHD1 interacts with OsHAP5C/D to delay heading date by inhibiting expression of Ehd1.'
            }
        ]
    },
    '2018.07': {
        name: '2018.07',
        list: [{
                TITLE: 'A large-scale whole-genome sequencing analysis reveals highly specific genome editing by both Cas9 and Cpf1 (Cas12a) nucleases in rice',
                AUTHOR: 'Yong Zhang Lab (University of Electronic Science and Technology of Chengdu, Chengdu , China; zhangyong916@uestc.edu.cn)',
                JOURNAL: 'Genome Biology',
                DOI: '10.1186/s13059-018-1458-5',
                ABSTRACT: 'Targeting specificity has been a barrier to applying genome editing systems in functional genomics, precise medicine and plant breeding. In plants, only limited studies have used whole-genome sequencing (WGS) to test off-target effects of Cas9. The cause of numerous discovered mutations is still controversial. Furthermore, WGS-based off-target analysis of Cpf1 (Cas12a) has not been reported in any higher organism to date. RESULTS: We conduct a WGS analysis of 34 plants edited by Cas9 and 15 plants edited by Cpf1 in T0 and T1 generations along with 20 diverse control plants in rice. The sequencing depths range from 45× to 105× with read mapping rates above 96%. Our results clearly show that most mutations in edited plants are created by the tissue culture process, which causes approximately 102 to 148 single nucleotide variations (SNVs) and approximately 32 to 83 insertions/deletions (indels) per plant. Among 12 Cas9 single guide RNAs (sgRNAs) and three Cpf1 CRISPR RNAs (crRNAs) assessed by WGS, only one Cas9 sgRNA resulted in off-target mutations in T0 lines at sites predicted by computer programs. Moreover, we cannot find evidence for bona fide off-target mutations due to continued expression of Cas9 or Cpf1 with guide RNAs in T1 generation. Our comprehensive and rigorous analysis of WGS data across multiple sample types suggests both Cas9 and Cpf1 nucleases are very specific in generating targeted DNA modifications and off-targeting can be avoided by designing guide RNAs with high specificity.'
            },
            {
                TITLE: 'Activities and specificities of CRISPR-Cas9 and Cas12a nucleases for targeted mutagenesis in maize',
                AUTHOR: 'Kan Wang Lab (Department of Agronomy, Iowa State University, Iowa, USA; kanwang@iastate.edu)',
                JOURNAL: 'Plant Biotechnology Journal',
                DOI: '10.1111/pbi.12982',
                ABSTRACT: 'CRISPR‐Cas9 and Cas12a (Cpf1) nucleases are two of the most powerful genome editing tools in plants. In this work, we compared their activities by targeting maize glossy2 gene coding region that has overlapping sequences recognized by both nucleases. We introduced constructs carrying SpCas9‐guide RNA (gRNA) and LbCas12a‐CRISPR RNA (crRNA) into maize inbred B104 embryos using Agrobacterium‐mediated transformation. On‐target mutation analysis showed that 90‐100% of the Cas9‐edited T0 plants carried indel mutations and 63‐77% of them were homozygous or biallelic mutants. In contrast, 0‐60% of Cas12a‐edited T0 plants had on‐target mutations. We then conducted CIRCLE‐seq analysis to identify genome‐wide potential off‐target sites for Cas9. A total of 18 and 67 potential off‐targets were identified for the two gRNAs, respectively, with an average of five mismatches compared to the target sites. Sequencing analysis of a selected subset of the off‐target sites revealed no detectable level of mutations in the T1 plants, which constitutively express Cas9 nuclease and gRNAs. In conclusion, our results suggest that the CRISPR‐Cas9 system used in this study is highly efficient and specific for genome editing in maize, while CRISPR‐Cas12a needs further optimization for improved editing efficiency.'
            },
            {
                TITLE: 'CRISPR/Cas9-mediated knockout of six glycosyltransferase genes in Nicotiana benthamiana for the production of recombinant proteins lacking β-1,2-xylose and core α-1,3-fucose',
                AUTHOR: 'Luisa Bortesi Lab (Department for Molecular Biotechnology, RWTH Aachen University, Aachen, Germany; luisa.bortesi@maastrichtuniversity.nl)',
                JOURNAL: 'Plant Biotechnology Journal',
                DOI: '10.1111/pbi.12981',
                ABSTRACT: 'Plants offer fast, flexible and easily scalable alternative platforms for the production of pharmaceutical proteins, but differences between plant and mammalian N‐linked glycans, including the presence of β‐1,2‐xylose and core α‐1,3‐fucose residues in plants, can affect the activity, potency and immunogenicity of plant‐derived proteins. Nicotiana benthamiana is widely used for the transient expression of recombinant proteins so it is desirable to modify the endogenous N‐glycosylation machinery to allow the synthesis of complex N‐glycans lacking β‐1,2‐xylose and core α‐1,3‐fucose. Here, we used multiplex CRISPR/Cas9 genome editing to generate N. benthamiana production lines deficient in plant‐specific α‐1,3‐fucosyltransferase and β‐1,2‐xylosyltransferase activity, reflecting the mutation of six different genes. We confirmed the functional gene knockouts by Sanger sequencing and mass spectrometry‐based N‐glycan analysis of endogenous proteins and the recombinant monoclonal antibody 2G12. Furthermore, we compared the CD64‐binding affinity of 2G12 glycovariants produced in wild‐type N. benthamiana, the newly generated FX‐KO line, and Chinese hamster ovary (CHO) cells, confirming that the glyco‐engineered antibody performed as well as its CHO‐produced counterpart.'
            },
            {
                TITLE:   'Simultaneous CRISPR/Cas9‐mediated editing of cassava eIF4E isoforms nCBP‐1 and nCBP‐2 reduces cassava brown streak disease symptom severity and incidence',
                AUTHOR:   'R. Bart Lab (Donald Danforth Plant Science Center, St. Louis MO, United States of America; rbart@danforthcenter.org)',
                JOURNAL: 'Plant Biotechnology Journal',
                DOI: '10.1111/pbi.12987  ',
                ABSTRACT: 'Cassava brown streak disease (CBSD) is a major constraint on cassava yields in East and Central Africa and threatens production in West Africa. CBSD is caused by two species of positive sense RNA viruses belonging to the family Potyviridae, genus Ipomovirus:Cassava brown streak  virus (CBSV) and Ugandan cassava brown streak virus (UCBSV). Diseases caused by the family Potyviridae require the interaction of viral genomelinked protein (VPg) and host eukaryotic translation initiation factor 4E (eIF4E) isoforms. Cassava encodes five eIF4E proteins: eIF4E, eIF(iso)4E-1, eIF(iso)4E-2, novel cap-binding protein-1 (nCBP-1), and nCBP-2. Protein-protein interaction experiments consistently found that VPg proteins associate with cassava nCBPs. CRISPR/Cas9-mediated genome editing was employed to generate ncbp-1, ncbp-2, and ncbp-1/ncbp-2 mutants in cassava cultivar 60444. Challenge with CBSV showed that ncbp-1/ncbp-2 mutants displayed delayed and attenuated CBSD aerial symptoms, as well as reduced severity and incidence of storage root necrosis. Suppressed disease symptoms were correlated with reduced virus titer in storage roots relative to wild-type controls. Our results demonstrate the ability to modify multiple genes simultaneously in cassava to achieve tolerance to CBSD. Future studies will investigate the contribution of remaining eIF4E isoforms on CBSD and translate this knowledge into an optimized strategy for protecting cassava from disease.'
            },
            {
                TITLE:   'Expanding the base editing scope in rice by using Cas9 variants',
                AUTHOR: 'Jiankang Zhu Lab (Shanghai Center for Plant Stress Biology, Chinese Academy of Sciences, Shanghai, China; jkzhu@sibs.ac.cn )',
                JOURNAL: 'Plant Biotechnology Journal',
                DOI: '10.1111/pbi.12993',
                ABSTRACT:   'Base editing is a novel genome editing strategy that enables irreversible base conversion at target loci without the need for double stranded break induction or homology-directed repair. Here, we developed new adenine and cytosine base editors with engineered SpCas9 and SaCas9 variants that substantially expand the targetable sites in the rice genome. These new base editors can edit endogenous genes in the rice genome with various efficiencies. Moreover, we show that adenine and cytosine base editing can be simultaneously executed in rice. The new base editors described here will be useful in rice functional genomics research and will advance precision molecular breeding in crops.'
            },
            {
                TITLE:   'Identifying a large number of high-yield genes in rice by pedigree analysis, whole-genome sequencing, and CRISPR-Cas9 gene knockout',
                AUTHOR:   'Dacheng Tian Lab (School of Life Sciences, Nanjing University, Nanjing, China; dtian@nju.edu.cn)',
                JOURNAL:   'PNAS',
                DOI:   '10.1073/pnas.1806110115',
                ABSTRACT:   'Repeated artificial selection of a complex trait facilitates the identification of genes underlying the trait, especially if multiple selected descendant lines are available. Here we developed a pedigree-based approach to identify genes underlying the Green Revolution (GR) phenotype. From a pedigree analysis, we selected 30 cultivars including the “miracle rice” IR8, a GR landmark, its ancestors and descendants, and also other related cultivars for identifying high-yield genes. Through sequencing of these genomes, we identified 28 ancestral chromosomal blocks that were maintained in all the high-yield cultivars under study. In these blocks, we identified six genes of known function, including the GR gene sd1, and 123 loci with genes of unknown function. We randomly selected 57 genes from the 123 loci for knockout or knockdown studies and found that a high proportion of these genes are essential or have phenotypic effects related to rice production. Notably, knockout lines have significant changes in plant height (P < 0.003), a key GR trait, compared with wild-type lines. Some gene knockouts or knockdowns were especially interesting. For example, knockout of Os10g0555100, a putative glucosyltransferase gene, showed both reduced growth and altered panicle architecture. In addition, we found that in some retained chromosome blocks several GR-related genes were clustered, although they have unrelated sequences, suggesting clustering of genes with similar functions. In conclusion, we have identified many high-yield genes in rice. Our method provides a powerful means to identify genes associated with a specific trait.'
            }

        ]
    },
    '2018.06': {
        name: '2018.06',
        list: [{
            TITLE: 'Expanded base editing in rice and wheat using a Cas9-adenosine deaminase fusion',
            AUTHOR: 'Caixia Gao Lab (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)',
            JOURNAL: 'Genome Biology',
            DOI: '10.1186/s13059-018-1443-z',
            ABSTRACT: 'Nucleotide base editors in plants have been limited to conversion of cytosine to thymine. Here, we describe a new plant adenine base editor based on an evolved tRNA adenosine deaminase fused to the nickase CRISPR/Cas9, enabling A•T to G•C conversion at frequencies up to 7.5% in protoplasts and 59.1% in regenerated rice and wheat plants. An endogenous gene is also successfully modified through introducing a gain-of-function point mutation to directly produce an herbicide-tolerant rice plant. With this new adenine base editing system, it is now possible to precisely edit all base pairs, thus expanding the toolset for precise editing in plants.'
        }, {
            TITLE: 'The CRISPR/Cas revolution reaches the RNA world: Cas13, a new Swiss Army knife for plant biologists',
            AUTHOR: 'Holger Puchta Lab (Karlsruhe Institute of Technology, Karlsruhe, Germany; holger.puchta@kit.edu)',
            JOURNAL: 'Plant Journal',
            DOI: '10.1111/tpj.13899',
            ABSTRACT: 'Application of the bacterial CRISPR/Cas systems to eukaryotes is revolutionizing biology. Cas9 and Cas12 (previously called Cpf1) are widely used as DNA nucleases for inducing site‐specific DNA breaks for different kinds of genome engineering applications, and in their mutated forms as DNA‐binding proteins to modify gene expression. Moreover, histone modifications, as well as cytosine methylation or base editing, were achieved with these systems in plants. Recently, with the discovery of the nuclease Cas13a (previously called C2c2), molecular biologists have obtained a system that enables sequence‐specific cleavage of single‐stranded RNA molecules. The latest experiments with this and also the alternative Cas13b system demonstrate that these proteins can be used in a similar manner in eukaryotes for RNA manipulation as Cas9 and Cas12 for DNA manipulations. The first application of Cas13a for post‐transcriptional regulation of gene expression in plants has been reported. Recent results show that the system is also applicable for combating viral infection in plants. As single‐stranded RNA viruses are by far the most abundant class of viruses in plants, the application of this system is of special promise for crops. More interesting applications are imminent for plant biologists, with nuclease dead versions of Cas13 enabling the ability to visualize RNA molecules in vivo, as well as to edit different kinds of RNA molecules at specific bases by deamination or to modify them by conjugation. Moreover, by combining DNA‐ and RNA‐directed systems, the most complex of changes in plant metabolism might be achievable.'
        }, {
            TITLE: 'CRISPR-FRT targets shared sites in a knock-out collection for off-the-shelf genome editing',
            AUTHOR: 'Jan Michiels Lab (University of Leuven, Leuven, Belgium; jan.michiels@kuleuven.vib.be)',
            JOURNAL: 'Nature Communications',
            DOI: '10.1038/s41467-018-04651-5',
            ABSTRACT: 'CRISPR advances genome engineering by directing endonuclease sequence specificity with a guide RNA molecule (gRNA). For precisely targeting a gene for modification, each genetic construct requires a unique gRNA. By generating a gRNA against the flippase recognition target (FRT) site, a common genetic element shared by multiple genetic collections, CRISPR-FRT circumvents this design constraint to provide a broad platform for fast, scarless, off-the-shelf genome engineering.'
        }]
    },
    '2018.05': {
        name: '2018.05',
        list: [{
            TITLE: 'Genotyping genome-edited mutations in plants using CRISPR ribonucleoprotein complexes',
            AUTHOR: 'Caixia Gao Lab (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.12938',
            ABSTRACT: 'Despite the great achievements in genome editing, accurately detecting mutations induced by sequence-specific nucleases is still a challenge in plants, especially in polyploidy plants. An efficient detection method is particularly vital when the mutation frequency is low or when a large population needs to be screened. Here, we applied purified CRISPR ribonucleoprotein complexes to cleave PCR products for genome-edited mutation detection in hexaploid wheat and diploid rice. We show that this mutation detection method is more sensitive than Sanger sequencing and more applicable than PCR/RE method without the requirement for restriction enzyme site. We also demonstrate that this detection method is especially useful for genome editing in wheat, because target sites are often surrounded by single nucleotide polymorphisms. By using this screening method, we were also able to detect foreign DNA-free tagw2 mutations induced by purified TALEN protein. Finally, we show that partial base editing mutations can also be detected using high fidelity SpCas9 variants or FnCpf1. The PCR/RNP method is low-cost and widely applicable for rapid detection of genome-edited mutation in plants. This article is protected by copyright.'
        }, {
            TITLE: 'Targeted mutagenesis in wheat microspores using CRISPR/Cas9',
            AUTHOR: 'Sateesh Kagale Lab (National Research Council Canada, Saskatoon, SK, Canada; sateesh.kagale@nrc-cnrc.gc.ca.)',
            JOURNAL: 'Scientific Reports',
            DOI: '10.1038/s41598-018-24690-8',
            ABSTRACT: 'CRISPR/Cas9 genome editing is a transformative technology that will facilitate the development of crops to meet future demands. However, application of gene editing is hindered by the long life cycle of many crop species and because desired genotypes generally require multiple generations to achieve. Single-celled microspores are haploid cells that can develop into double haploid plants and have been widely used as a breeding tool to generate homozygous plants within a generation. In this study, we combined the CRISPR/Cas9 system with microspore technology and developed an optimized haploid mutagenesis system to induce genetic modifications in the wheat genome. We investigated a number of factors that may affect the delivery of CRISPR/Cas9 reagents into microspores and found that electroporation of a minimum of 75,000 cells using 10-20 µg DNA and a pulsing voltage of 500 V is optimal for microspore transfection using the Neon transfection system. Using multiple Cas9 and sgRNA constructs, we present evidence for the seamless introduction of targeted modifications in an exogenous DsRed gene and two endogenous wheat genes, including TaLox2 and TaUbiL1. This study demonstrates the value and feasibility of combining microspore technology and CRISPR/Cas9-based gene editing for trait discovery and improvement in plants.'
        }, {
            TITLE: 'Genome Editing: Targeting Susceptibility Genes for Plant Disease Resistance',
            AUTHOR: 'Shahid Mansoor Lab (National Institute for Biotechnology and Genetic Engineering, Faisalabad, Pakistan; shahidmansoor7@gmail.com)',
            JOURNAL: 'Trends in Biotechnology',
            DOI: '10.1016/j.tibtech.2018.04.005',
            ABSTRACT: 'Plant pathogens pose a major threat to crop productivity. Typically, phytopathogens exploit plants\' susceptibility (S) genes to facilitate their proliferation. Disrupting these S genes may interfere with the compatibility between the host and the pathogens and consequently provide broad-spectrum and durable disease resistance. In the past, genetic manipulation of such S genes has been shown to confer disease resistance in various economically important crops. Recent studies have accomplished this task in a transgene-free system using new genome editing tools, including clustered regularly interspaced palindromic repeats (CRISPR). In this Opinion article, we focus on the use of genome editing to target S genes for the development of transgene-free and durable disease-resistant crop varieties.'
        }, {
            TITLE: 'CRISPR/Cas9-mediated gene targeting in Arabidopsis using sequential transformation',
            AUTHOR: 'Jiankang Zhu Lab (Purdue University, West Lafayette, IN, USA; jkzhu@sibs.ac.cn)',
            JOURNAL: 'Nature Communications',
            DOI: '10.1038/s41467-018-04416-0',
            ABSTRACT: 'Homologous recombination-based gene targeting is a powerful tool for precise genome modification and has been widely used in organisms ranging from yeast to higher organisms such as Drosophila and mouse. However, gene targeting in higher plants, including the most widely used model plant Arabidopsis thaliana, remains challenging. Here we report a sequential transformation method for gene targeting in Arabidopsis. We find that parental lines expressing the bacterial endonuclease Cas9 from the egg cell- and early embryo-specific DD45 gene promoter can improve the frequency of single-guide RNA-targeted gene knock-ins and sequence replacements via homologous recombination at several endogenous sites in the Arabidopsis genome. These heritable gene targeting can be identified by regular PCR. Our approach enables routine and fine manipulation of the Arabidopsis genome.'
        }, {
            TITLE: 'Multiplex gene editing in rice with simplified CRISPR-Cpf1 and CRISPR-Cas9 systems',
            AUTHOR: 'Jiankang Zhu Lab (Purdue University, West Lafayette, IN, USA; jkzhu@sibs.ac.cn)',
            JOURNAL: 'Journal of Integrative Plant Biology',
            DOI: '10.1111/jipb.12667',
            ABSTRACT: 'We developed simplified single transcriptional unit (SSTU) CRISPR systems for multiplex gene editing in rice using FnCpf1, LbCpf1 or Cas9, in which the nuclease and its crRNA array are co-expressed from a single Pol II promoter, without any additional processing machinery. Our SSTU systems are easy to construct and effective in mediating multiplex genome editing.'
        }, {
            TITLE: 'High-throughput detection and screening of plants modified by gene editing using quantitative real-time PCR',
            AUTHOR: 'Junfeng Xu Lab (Institute of Quality and Standard for Agro-Products, Zhejiang Academy of Agricultural Sciences, Hangzhou, China; njjfxu@163.com)',
            JOURNAL: 'Plant Journal',
            DOI: '10.1111/tpj.13961',
            ABSTRACT: 'Gene editing techniques are becoming powerful tools for modifying target genes in organisms. Although several methods have been developed to detect gene-edited organisms, these techniques are time- and labour-intensive. Meanwhile, few studies have investigated high-throughput detection and screening strategies for plants modified by gene editing. In this study, we developed a simple, sensitive and high-throughput quantitative real-time(qPCR)-based method. The qPCR-based method exploits two differently labelled probes that are placed within one amplicon at the gene editing target site to simultaneously detect the wild type and a gene-edited mutant. We showed that the qPCR-based method can accurately distinguish CRISPR/Cas9-induced mutants from the wild type in several different plant species, such as Oryza sativa, Arabidopsis thaliana, Sorghum bicolor, and Zea mays. Moreover, the method can subsequently determine the mutation type by direct sequencing of the qPCR products of mutations due to gene editing. The qPCR-based method is also sufficiently sensitive to distinguish between heterozygous and homozygous mutations in T0 transgenic plants. In a 384-well plate format, the method enabled the simultaneous analysis of up to 128 samples in three replicates without handling the post-PCR products. Thus, we propose that our method is an ideal choice for screening plants modified by gene editing from many candidates in T0 transgenic plants, which will be widely used in the area of plant gene editing.'
        }, {
            TITLE: 'Efficient in planta gene targeting in Arabidopsis using egg cell‐specific expression of the Cas9 nuclease of Staphylococcus aureus',
            AUTHOR: 'Holger Puchta Lab (Karlsruhe Institute of Technology, Karlsruhe, Germany; holger.puchta@kit.edu)',
            JOURNAL: 'Plant Journal',
            DOI: '10.1111/tpj.13893',
            ABSTRACT: 'Gene targeting (GT), the programmed change of genomic sequences by homologous recombination (HR), is still a major challenge in plants. We previously developed an in planta GT strategy by simultaneously releasing from the genome a dsDNA donor molecule and creating a double‐stranded break (DSB) at a specific site within the targeted gene. Using Cas9 form Streptococcus pyogenes (SpCas9) under the control of a ubiquitin gene promoter, we obtained seeds harbouring GT events, although at a low frequency. In the present research we tested different developmentally controlled promotors and different kinds of DNA lesions for their ability to enhance GT of the acetolactate synthase (ALS) gene of Arabidopsis. For this purpose, we used Staphylococcus aureus Cas9 (SaCas9) nuclease and the SpCas9 nickase in various combinations. Thus, we analysed the effect of single‐stranded break (SSB) activation of a targeted gene and/or the HR donor region. Moreover, we tested whether DSBs with 5′ or 3′ overhangs can improve in planta GT. Interestingly, the use of the SaCas9 nuclease controlled by an egg cell‐specific promoter was the most efficient: depending on the line, in the very best case 6% of all seeds carried GT events. In a third of all lines, the targeting occurred around the 1% range of the tested seeds. Molecular analysis revealed that in about half of the cases perfect HR of both DSB ends occurred. Thus, using the improved technology, it should now be feasible to introduce any directed change into the Arabidopsis genome at will.'
        }, {
            TITLE: 'Engineering herbicide-resistant watermelon variety through CRISPR/Cas9-mediated base-editing',
            AUTHOR: 'Yong Xu Lab (Beijing Academy of Agriculture and Forestry Sciences, Beijing, China; xuyong@nercv.org)',
            JOURNAL: 'Plant Cell Reports',
            DOI: '10.1007/s00299-018-2299-0',
            ABSTRACT: 'In this study, we successfully established high-efficient base-editing system and created non-GM herbicide-resistant watermelon varieties. Because these transgene-free baseedited herbicide-resistant watermelon plants are genetically identical to those bred via traditional mutagenesis, no extra regulations should be applied to these transgene-free base-edited mutants. Without fitness cost associated with the base-edited point mutations, this non-GM herbicide-resistant watermelon is now ready for immediate field application. Our report provides the first solid example showing that the CRISPR/Cas9-mediated base-editing system is a tremendously powerful tool for improvement of watermelon varieties.'
        }, {
            TITLE: 'The rice blast resistance gene Ptr encodes an atypical protein required for broad-spectrum disease resistance',
            AUTHOR: 'Yulin Jia Lab (USDA ARS Dale Bumpers National Rice Research Center, Stuttgart, AR, USA; jia@ars.usda.gov)',
            JOURNAL: 'Nature Communications',
            DOI: '10.1038/s41467-018-04369-4',
            ABSTRACT: 'Plant resistance genes typically encode proteins with nucleotide binding site-leucine rich repeat (NLR) domains. Here we show that Ptr is an atypical resistance gene encoding a protein with four Armadillo repeats. Ptr is required for broad-spectrum blast resistance mediated by the NLR R gene Pi-ta and by the associated R gene Pi-ta2. Ptr is expressed constitutively and encodes two isoforms that are mainly localized in the cytoplasm. A two base pair deletion within the Ptr coding region in the fast neutron-generated mutant line M2354 creates a truncated protein, resulting in susceptibility to M. oryzae. Targeted mutation of Ptr in a resistant cultivar using CRISPR/Cas9 leads to blast susceptibility, further confirming its resistance function. The cloning of Ptr may aid in the development of broad spectrum blast resistant rice.'
        }]
    },
    '2018.04': {
        name: '2018.04',
        list: [{
            TITLE: 'Base editing with a Cpf1–cytidine deaminase fusion',
            AUTHOR: 'Jia Chen Lab (Shanghai Tech University, Shanghai, China; chenjia@shanghaitech.edu.cn)',
            JOURNAL: 'Nature Biotechnology',
            DOI: '10.1038/nbt.4102',
            ABSTRACT: 'The targeting range of CRISPR–Cas9 base editors (BEs) is limited by their G/C-rich protospacer-adjacent motif (PAM) sequences. To overcome this limitation, we developed a CRISPR–Cpf1-based BE by fusing the rat cytosine deaminase APOBEC1 to a catalytically inactive version of Lachnospiraceae bacterium Cpf1. The base editor recognizes a T-rich PAM sequence and catalyzes C-to-T conversion in human cells, while inducing low levels of indels, non-C-to-T substitutions and off-target editing.'
        }, {
            TITLE: 'Precise Cas9 targeting enables genomic mutation prevention',
            AUTHOR: 'George Church Lab (Wyss Institute for Biologically Inspired Engineering, Harvard University, Cambridge, MA, USA; ac4304@cumc.columbia.edu)',
            JOURNAL: 'PNAS',
            DOI: '10.1073/pnas.1718148115',
            ABSTRACT: 'Here, we present a generalized method of guide RNA “tuning” that enables Cas9 to discriminate between two target sites that differ by a single-nucleotide polymorphism. We employ our methodology to generate an in vivo mutation prevention system in which Cas9 actively restricts the occurrence of undesired gain-of-function mutations within a population of engineered organisms. We further demonstrate that the system is scalable to a multitude of targets and that the general tuning and prevention concepts are portable across engineered Cas9 variants and Cas9 orthologs. Finally, we show that the mutation prevention system maintains robust activity even when placed within the complex environment of the mouse gastrointestinal tract.'
        }, {
            TITLE: 'CRISPys: Optimal sgRNA design for editing multiple members of a gene family using the CRISPR system',
            AUTHOR: 'Itay Mayrose Lab (Tel Aviv University, Ramat Aviv, Israel; itaymay@post.tau.ac.il)',
            JOURNAL: 'Journal of Molecular Biology',
            DOI: '10.1016/j.jmb.2018.03.019',
            ABSTRACT: 'The development of the CRISPR-Cas9 system in recent years has made eukaryotic genome editing, and specifically gene knockout for reverse genetics, a simple and effective task. The system is directed to a genomic target site by a programmed single-guide RNA (sgRNA) that base-pairs with it, subsequently leading to site-specific modifications. However, many gene families in eukaryotic genomes exhibit partially overlapping functions and, thus, the knockout of one gene might be concealed by the function of the other. In such cases, the reduced specificity of the CRISPR-Cas9 system, which may lead to the modification of genomic sites that are not identical to the sgRNA, can be harnessed for the simultaneous knockout of multiple homologous genes. We introduce CRISPys, an algorithm for the optimal design of sgRNAs that would potentially target multiple members of a given gene family. CRISPys first clusters all the potential targets in the input sequences into a hierarchical tree structure that specifies the similarity among them. Then, sgRNAs are proposed in the internal nodes of the tree by embedding mismatches where needed, such that the efficiency to edit the induced targets is maximized. We suggest several approaches for designing the optimal individual sgRNA and an approach to compute the optimal set of sgRNAs for cases when the experimental platform allows for more than one. The latter may optionally account for the homologous relationships among gene-family members. We further show that CRISPys outperforms simpler alignment-based techniques by in-silico examination over all gene families in the Solanum lycopersicum genome.'
        }, {
            TITLE: 'Efficient allelic replacement in rice by gene editing: a case study of the NRT1.1B gene',
            AUTHOR: 'Lanqin Xia Lab (Institute of Crop Sciences, Chinese Academy of Agricultural Sciences, Beijing, China; xialanqin@caas.cn)',
            JOURNAL: 'Journal of Integrative Plant Biology',
            DOI: '10.1111/jipb.12650',
            ABSTRACT: 'Precise replacement of an existing allele in commercial cultivars with an elite allele is a major goal in crop breeding. A single nucleotide polymorphism in NRT1.1B gene between japonica and indica rice is responsible for the improved nitrogen use efficiency in indica rice. Herein, we precisely replaced the japonica NRT1.1B allele with the indica allele in just one generation using CRISPR/Cas9 gene editing technology. No additional selective pressure was needed to enrich the precise replacement events. This work demonstrates the feasibility of replacing any genes with elite alleles within one generation, greatly expanding our ability to improve agriculturally important traits.'
        }, {
            TITLE: 'Incorporation of bridged nucleic acids into CRISPR RNAs improves Cas9 endonuclease specificity',
            AUTHOR: 'Basil Hubbard Lab (University of Alberta, Edmonton, AB, Canada; bphubbard@ualberta.ca)',
            JOURNAL: 'Nature Communications',
            DOI: '10.1038/s41467-018-03927-0',
            ABSTRACT: 'Off-target DNA cleavage is a paramount concern when applying CRISPR-Cas9 gene-editing technology to functional genetics and human therapeutic applications. Here, we show that incorporation of next-generation bridged nucleic acids (2\',4\'-BNANC[N-Me]) as well as locked nucleic acids (LNA) at specific locations in CRISPR-RNAs (crRNAs) broadly reduces off-target DNA cleavage by Cas9 in vitro and in cells by several orders of magnitude. Using single-molecule FRET experiments we show that BNANC incorporation slows Cas9 kinetics and improves specificity by inducing a highly dynamic crRNA-DNA duplex for off-target sequences, which shortens dwell time in the cleavage-competent, "zipped" conformation. In addition to describing a robust technique for improving the precision of CRISPR/Cas9-based gene editing, this study illuminates an application of synthetic nucleic acids.'
        }, {
            TITLE: 'Further analysis of barley MORC1 using a highly efficient RNA-guided Cas9 gene editing system',
            AUTHOR: 'Karl-Heinz Kogel Lab (Justus-Liebig University Giessen, Giessen, Germany; karl-heinz.kogel@agrar.uni-giessen.de)',
            JOURNAL: 'Plant Biotechnology Journal',
            DOI: '10.1111/pbi.12924',
            ABSTRACT: 'Microrchidia (MORC) proteins comprise a family of proteins that have been identified in prokaryotes and eukaryotes. They are defined by two hallmark domains: a GHKL-type ATPase and an S5 fold. In plants, MORC proteins were first discovered in a genetic screen for Arabidopsis thaliana mutants compromised for resistance to a viral pathogen. Subsequent studies expanded their role in plant immunity and revealed their involvement in gene silencing and genome stabilization. Little is known about the role of MORC proteins of cereals, especially because knock-out (KO) mutants were not available and assessment of loss of function relied only on RNAi strategies, which were arguable, given that MORC proteins in itself are influencing gene silencing. Here we used a Streptococcus pyogenes Cas9 (SpCas9)-mediated KO strategy to functionally study HvMORC1, one of the currently seven MORC members of barley. Using a novel barley RNA Pol III-dependent U3 small nuclear RNA (snRNA) promoter to drive expression of the synthetic single guide RNA (sgRNA), we achieved a very high mutation frequency in HvMORC1. High frequencies of mutations were detectable by target sequencing in the callus, the T0 generation (77%) and T1 (70%-100%), which constitutes an important improvement of the gene editing technology in cereals. Corroborating and extending earlier findings, SpCas9-edited hvmorc1-KO barley, in clear contrast to Arabidopsis atmorc1 mutants, had a distinct phenotype of increased disease resistance to fungal pathogens, while morc1 mutants of either plant showed de-repressed expression of transposable elements (TEs), substantiating that plant MORC proteins contribute to genome stabilization in mono- and dicotyledonous plants.'
        }]
    },
    '2018.03': {
        name: '2018.03',
        list: [{
                TITLE: 'Precision genome engineering through adenine and cytosine base editing',
                AUTHOR: 'Jin-Soo Kim (Seoul National University, Seoul, Republic of Korea; jskim01@snu.ac.kr)',
                JOURNAL: 'Nature Plants',
                doi: '10.1038/s41477-018-0115-z',
                ABSTRACT: 'Adenine base editors (ABEs), composed of an engineered deaminase and a catalytically impaired CRISPR-Cas9 variant, are powerful new tools for targeted base editing in cells and organisms. Together with cytosine base editors (CBEs), ABEs enable single-nucleotide conversions cleanly, efficiently and reversibly without double-stranded DNA cleavage, advancing genome editing in a new dimension.'
            },
            {
                TITLE: 'A highly specific SpCas9 variant is identified by in vivo screening in yeast',
                AUTHOR: 'Anna Cereseto Lab (University of Trento, Trento, Italy)',
                JOURNAL: 'Nature Biotechnology',
                doi: '10.1038/nbt.4066',
                ABSTRACT: 'Despite the utility of CRISPR–Cas9 nucleases for genome editing, the potential for off-target activity limits their application, especially for therapeutic purposes1,2. We developed a yeast-based assay to identify optimized Streptococcus pyogenes Cas9 (SpCas9) variants that enables simultaneous evaluation of on- and off-target activity. We screened a library of SpCas9 variants carrying random mutations in the REC3 domain and identified mutations that increased editing accuracy while maintaining editing efficiency. We combined four beneficial mutations to generate evoCas9, a variant that has fidelity exceeding both wild-type (79-fold improvement) and rationally designed Cas9 variants3,4 (fourfold average improvement), while maintaining near wild-type on-target editing efficiency (90% median residual activity). Evaluating evoCas9 on endogenous genomic loci, we demonstrated a substantially improved specificity and observed no off-target sites for four of the eight single guide RNAs (sgRNAs) tested. Finally, we showed that following long-term expression (40 d), evoCas9 strongly limited the nonspecific cleavage of a difficult-to-discriminate off-target site and fully abrogated the cleavage of two additional off-target sites.'
            },
            {
                TITLE: 'Efficient DNA-free genome editing of bread wheat using CRISPR/Cas9 ribonucleoprotein complexes',
                AUTHOR: 'Caixia Gao Lab (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)',
                JOURNAL: 'Nature Communications',
                doi: '10.1038/ncomms14261',
                ABSTRACT: 'Substantial efforts are being made to optimize the CRISPR/Cas9 system for precision crop breeding. The avoidance of transgene integration and reduction of off-target mutations are the most important targets for optimization. Here, we describe an efficient genome editing method for bread wheat using CRISPR/Cas9 ribonucleoproteins (RNPs). Starting from RNP preparation, the whole protocol takes only seven to nine weeks, with four to five independent mutants produced from 100 immature wheat embryos. Deep sequencing reveals that the chance of off-target mutations in wheat cells is much lower in RNP mediated genome editing than in editing with CRISPR/Cas9 DNA. Consistent with this finding, no off-target mutations are detected in the mutant plants. Because no foreign DNA is used in CRISPR/Cas9 RNP mediated genome editing, the mutants obtained are completely transgene free. This method may be widely applicable for producing genome edited crop plants and has a good prospect of being commercialized.'
            },
            {
                TITLE: 'High efficiency genome editing using a dmc1 promoter-controlled CRISPR/Cas9 system in maize',
                AUTHOR: 'Fangpu Han Lab (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China; fphan@genetics.ac.cn)',
                JOURNAL: 'Plant Biotechnology Journal',
                DOI: '10.1111/pbi.12920',
                ABSTRACT: 'Previous studies revealed that the promoters for driving both Cas9 and sgRNAs are quite important for efficient genome editing by CRISPR/Cas9 in plants. Here we report our results of targeted genome editing using the maize dmc1 gene promoter combined with the U3 promoter for Cas9 and sgRNA, respectively. Three loci in the maize genome were selected for targeting. The T0 plants regenerated were highly-efficiently edited at the target sites with homozygous or bi-allelic mutants accounting for about 66%. The mutations in T0 plants could be stably transmitted to the T1 generation and new mutations could be generated in gametes or zygotes. Whole genome re-sequencing indicated that no off-target mutations could be detected in the predicted loci with sequence similarity to the targeted site. Our results show that the dmc1 promoter-controlled (DPC) CRISPR/Cas9 system is highly efficient in maize and provide further evidence that the optimization of the promoters used for the CRISPR/Cas9 system is important for enhancing the efficiency of targeted genome editing in plants. The evolutionary conservation of the dmc1 gene suggests its potential for use in other plant species. This article is protected by copyright. '
            },
            {
                TITLE: 'Genome editing in potato via CRISPR-Cas9 ribonucleoprotein delivery',
                AUTHOR: 'Mariette Andersson Lab (Swedish University of Agricultural Sciences, Alnarp, Sweden; mariette.andersson@slu.se)',
                JOURNAL: 'Physiologia Plantarum',
                DOI: '10.1111/ppl.12731',
                ABSTRACT: 'CRISPR-Cas9 can be used as an efficient tool for genome editing in potato (Solanum tuberosum). From both a scientific and a regulatory perspective, it is beneficial if integration of DNA in the potato genome is avoided. We have implemented a DNA-free genome editing method, using delivery of CRISPR-Cas9 ribonucleoproteins (RNP) to potato protoplasts, by targeting the gene encoding a granule bound starch synthase (GBSS, EC 2.4.1.242). The RNP method was directly implemented using previously developed protoplast isolation, transfection and regeneration protocols without further adjustments. Cas9 protein was preassembled with RNA produced either synthetically or by in vitro transcription. RNP with synthetically produced RNA (cr-RNP) induced mutations, i.e. indels, at a frequency of up to 9%, with all mutated lines being transgene-free. A mutagenesis frequency of 25% of all regenerated shoots was found when using RNP with in vitro transcriptionally produced RNA (IVT-RNP)h However, more than 80% of the shoots with confirmed mutations had unintended inserts in the cut site, which was in the same range as when using DNA delivery. The inserts originated both from DNA template remnants from the in vitro transcription, and from chromosomal potato DNA. In 2 - 3% of the regenerated shoots from the RNP-experiments, mutations were induced in all four alleles resulting in a complete knockout of the GBSS enzyme function.'
            },


            {
                TITLE: 'CRISPR/Cas13 as a Tool for RNA Interference',
                AUTHOR: 'Magdy Mahfouz Lab (King Abdullah University of Science and Technology, Thuwal, Saudi Arabia; magdy.mahfouz@kaust.edu.sa)',
                JOURNAL: 'Trends in Plant Science',
                DOI: '10.1016/j.tplants.2018.03.003',
                ABSTRACT: 'Almost all biological processes involve RNA, making it crucial to develop tools for manipulation of the transcriptome. The bacterial CRISPR/Cas13 system was recently rewired to facilitate RNA manipulation in eukaryotes, including plants. We discuss here the opportunities and limitations of using CRISPR/Cas13 in plants for various types of RNA manipulation.'
            },
            {
                TITLE: 'Low-gluten, nontransgenic wheat engineered with CRISPR/Cas9',
                AUTHOR: 'Francisco Barro Lab (Instituto de Agricultura Sostenible, Córdoba, Spain; fbarro@ias.csic.es)',
                JOURNAL: 'Plant Biotechnology Journal',
                DOI: '10.1111/pbi.12837',
                ABSTRACT: 'Coeliac disease is an autoimmune disorder triggered in genetically predisposed individuals by the ingestion of gluten proteins from wheat, barley and rye. The α-gliadin gene family of wheat contains four highly stimulatory peptides, of which the 33-mer is the main immunodominant peptide in patients with coeliac. We designed two sgRNAs to target a conserved region adjacent to the coding sequence for the 33-mer in the α-gliadin genes. Twenty-one mutant lines were generated, all showing strong reduction in α-gliadins. Up to 35 different genes were mutated in one of the lines of the 45 different genes identified in the wild type, while immunoreactivity was reduced by 85%. Transgene-free lines were identified, and no off-target mutations have been detected in any of the potential targets. The low-gluten, transgene-free wheat lines described here could be used to produce low-gluten foodstuff and serve as source material to introgress this trait into elite wheat varieties.'
            },
            {
                TITLE: 'CRISPR/Cas9-mediated efficient targeted mutagenesis in grape in the first generation',
                AUTHOR: 'Xiping Wang Lab (Northwest A&F University, Yangling, Shaanxi, China; wangxiping@nwsuaf.edu.cn)',
                JOURNAL: 'Plant Biotechnology Journal',
                DOI: '10.1111/pbi.12832',
                ABSTRACT: 'The clustered regularly interspaced short palindromic repeats-associated protein 9 (CRISPR/Cas9) system is a powerful tool for editing plant genomes. Efficient genome editing of grape (Vitis vinifera) suspension cells using the type II CRISPR/Cas9 system has been demonstrated; however, it has not been established whether this system can be applied to get biallelic mutations in the first generation of grape. In this current study, we designed four guide RNAs for the VvWRKY52 transcription factor gene for using with the CRISPR/Cas9 system, and obtained transgenic plants via Agrobacterium-mediated transformation, using somatic embryos of the Thompson Seedless cultivar. Analysis of the first-generation transgenic plants verified 22 mutant plants of the 72 T-DNA-inserted plants. Of these, 15 lines carried biallelic mutations and seven were heterozygous. A range of RNA-guided editing events, including large deletions, were found in the mutant plants, while smaller deletions comprised the majority of the detected mutations. Sequencing of potential off-target sites for all four targets revealed no off-target events. In addition, knockout of VvWRKY52 in grape increased the resistance to Botrytis cinerea. We conclude that the CRISPR/Cas9 system allows precise genome editing in the first generation of grape and represents a useful tool for gene functional analysis and grape molecular breeding.'
            }
        ]
    },
    '2018.02': {
        name: '2018.02',
        list: [{
                TITLE: 'Genome editing of bread wheat using biolistic delivery of CRISPR/Cas9 in vitro transcripts or ribonucleoproteins',
                AUTHOR: 'Caixia Gao Lab (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)',
                JOURNAL: 'Nature Protocols',
                doi: '10.1038/nprot.2017.145',
                ABSTRACT: 'This protocol is an extension to: Nature Protocols 9, 2395-2410 (2014); doi:10.1038/nprot.2014.157; published online 18 September 2014. In recent years, CRISPR/Cas9 has emerged as a powerful tool for improving crop traits. Conventional plant genome editing mainly relies on plasmid-carrying cassettes delivered by Agrobacterium or particle bombardment. Here, we describe DNA-free editing of bread wheat by delivering in vitro transcripts (IVTs) or ribonucleoprotein complexes (RNPs) of CRISPR/Cas9 by particle bombardment. This protocol serves as an extension of our previously published protocol on genome editing in bread wheat using CRISPR/Cas9 plasmids delivered by particle bombardment. The methods we describe not only eliminate random integration of CRISPR/Cas9 into genomic DNA, but also reduce off-target effects. In this protocol extension article, we present detailed protocols for preparation of IVTs and RNPs; validation by PCR/restriction enzyme (RE) and next-generation sequencing; delivery by biolistics; and recovery of mutants and identification of mutants by pooling methods and Sanger sequencing. To use these protocols, researchers should have basic skills and experience in molecular biology and biolistic transformation. By using these protocols, plants edited without the use of any foreign DNA can be generated and identified within 9-11 weeks.'
            },
            {
                TITLE: 'CRISPR/Cas9: An RNA-guided highly precise synthetic tool for plant genome editing',
                AUTHOR: 'Turgay Unver Lab (Izmir International Biomedicine and Genome Institute, Dokuz Eylul University, Izmir, Turkey; turgay.unver@deu.edu.tr)',
                JOURNAL: 'Journal of Cellular Physiology',
                doi: '10.1002/jcp.25970',
                ABSTRACT: 'CRISPR/Cas9 is a newly developed and naturally occurred genome editing tool, which is originally used by bacteria for immune defence. In the past years, it has been quickly employed and modified to precisely edit genome sequences in both plants and animals. Compared with the well-developed zinc finger nucleases (ZFNs) and transcription activator-like effector nucleases (TALENs), CRISPR/Cas9 has lots of advantages, including easier to design and implement, higher targeting efficiency, and less expensive. Thus, it is becoming one of the most powerful tools for knockout of an individual gene as well as insertion of one gene and/or control of gene transcription. Studies have shown that CRISPR/Cas9 is a great tool to edit many genes in a variety of plant species, including the model plant species as well as agriculturally important crops, such as cotton, maize, wheat, and rice. CRISPR/Cas9-based genome editing can be used for plant functional studies and plant improvement to yield, quality, and tolerance to environmental stress.'
            },
            {
                TITLE: 'High-efficient A·T to G·C base editing by Cas9n-guided tRNA adenosine deaminase in rice',
                AUTHOR: 'Huanbin Zhou Lab (Institute of Plant Protection, Chinese Academy of Agricultural Sciences, Beijing, China; hbzhou@ippcaas.cn)',
                JOURNAL: 'Molecular Plant',
                doi: '10.1016/j.molp.2018.02.008',
                ABSTRACT: 'The spontaneous deamination of cytosine is a major source of transitions from C•G to T•A base pairs, which account for half of known pathogenic point mutations in humans. The ability to efficiently convert targeted A•T base pairs to G•C could therefore advance the study and treatment of genetic diseases. The deamination of adenine yields inosine, which is treated as guanine by polymerases, but no enzymes are known to deaminate adenine in DNA. Here we describe adenine base editors (ABEs) that mediate the conversion of A•T to G•C in genomic DNA. We evolved a transfer RNA adenosine deaminase to operate on DNA when fused to a catalytically impaired CRISPR–Cas9 mutant. Extensive directed evolution and protein engineering resulted in seventh-generation ABEs that convert targeted A•T base pairs efficiently to G•C (approximately 50% efficiency in human cells) with high product purity (typically at least 99.9%) and low rates of indels (typically no more than 0.1%). ABEs introduce point mutations more efficiently and cleanly, and with less off-target genome modification, than a current Cas9 nuclease-based method, and can install disease-correcting or disease-suppressing mutations in human cells. Together with previous base editors, ABEs enable the direct, programmable introduction of all four transition mutations without double-stranded DNA cleavage.'
            },
            {
                TITLE: 'Engineering introns to express RNA guides for Cas9- and Cpf1-mediated multiplex genome editing',
                AUTHOR: 'Kabin Xie Lab (Huazhong Agricultural University, Wuhan, China; kabinxie@mail.hzau.edu.cn)',
                JOURNAL: 'Molecular Plant',
                doi: '10.1016/j.molp.2018.02.005',
                ABSTRACT: 'The Clustered Regularly Interspaced Short Palindromic Repeat (CRISPR) system has emerged as the revolutionary platform for DNA targeting. This system uses a site-specific RNA guide to direct a CRISPR effector (e.g., Cas9 and Cpf1) to a DNA target. Here, we elaborate a general strategy to simultaneously express multiple guide RNAs (gRNA) and CRISPR RNAs (crRNA) from introns of Cas9 and Cpf1. This method utilizes the endogenous tRNA processing system or crRNA processing activity of Cpf1 to cleave the spliced intron which contains tRNA-gRNA polycistron or crRNA-crRNA array. We demonstrated that the tRNA-gRNA intron is able to fuse with Cas9 as one gene. Such hybrid gene could be expressed using one polymerase II promoter and exhibited high efficiency and robustness to simultaneously targeting multiple sites. We also implemented this strategy in Cpf1-mediated genome editing using intronic tRNA-crRNA and crRNA-crRNA array. Interestingly, hybrid genes containing Cpf1 and intronic crRNA array exhibited remarkably increased efficiency than the conventional Cpf1 vectors. Taken together, this study presents a method to express CRISPR reagents from one hybrid gene to increase genome editing efficiency and capacity. Owing to its simplicity and versatility, this method could be broadly used to develop sophisticated CRISPR tools in eukaryotes.'
            },
            {
                TITLE: 'The future of CRISPR technologies in agriculture',
                AUTHOR: 'Caixia Gao (Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China; cxgao@genetics.ac.cn)',
                JOURNAL: 'Nature Reviews Molecular Cell Biology',
                doi: '10.1038/nrm.2018.2',
                ABSTRACT: 'Conventional plant breeding is unlikely to meet increasing food demands and other environmental challenges. By contrast, CRISPR technology is erasing barriers to genome editing and could revolutionize plant breeding. However, to fully benefit from the CRISPR revolution, we should focus on resolving its technical and regulatory uncertainties.'
            }
        ]
    }
}